-- MySQL dump 10.13  Distrib 5.5.42, for osx10.10 (x86_64)
--
-- Host: localhost    Database: metaDB
-- ------------------------------------------------------
-- Server version	5.5.42

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `accession`
--

DROP TABLE IF EXISTS `accession`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `accession` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `randomcode` varchar(50) NOT NULL,
  `name` varchar(50) NOT NULL,
  `email` varchar(50) NOT NULL,
  `institution` varchar(50) NOT NULL,
  `password` varchar(50) NOT NULL,
  `downloadedtimes` int(11) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `accession`
--

LOCK TABLES `accession` WRITE;
/*!40000 ALTER TABLE `accession` DISABLE KEYS */;
INSERT INTO `accession` VALUES (1,'111','Zhengyu Guo','a@gmail.com','Guo','1234567',0);
/*!40000 ALTER TABLE `accession` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `disease`
--

DROP TABLE IF EXISTS `disease`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `disease` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `disease` varchar(50) DEFAULT NULL,
  `diseaseMesh` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=32 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `disease`
--

LOCK TABLES `disease` WRITE;
/*!40000 ALTER TABLE `disease` DISABLE KEYS */;
INSERT INTO `disease` VALUES (1,'E-GEOD-61216','',''),(2,'E-GEOD-60968','inflammation','D007249'),(3,'E-GEOD-59215','',''),(4,'E-GEOD-58996','Huntington\'s Disease','D006816'),(5,'E-GEOD-58973','inflammation','D007249'),(6,'E-GEOD-58972','inflammation','D007249'),(7,'E-GEOD-58971','inflammation','D007249'),(8,'E-GEOD-58905','',''),(9,'E-GEOD-58498','DiGeorge Syndrome','D004062'),(10,'E-GEOD-57959','Age related Cataract','C563812'),(11,'E-GEOD-57837','inflammation','D007249'),(12,'E-GEOD-57396','',''),(13,'E-GEOD-57147','acute myeloid leukemia','D015470'),(14,'E-GEOD-56127','Neuroaxonal Dystrophies','D019150'),(15,'E-GEOD-56018','',''),(16,'E-GEOD-55377','retinal degeneration','D012162'),(17,'E-GEOD-55376','retinal degeneration','D012162'),(18,'E-GEOD-54631','',''),(19,'E-GEOD-54154','Diabetes Mellitus, Type 2','D003924'),(20,'E-GEOD-51008','neoplasms','D009369'),(21,'E-GEOD-49949','Mowat-Wilson syndrome','C536990'),(22,'E-GEOD-49555','Hepatocellular Carcinoma','D006528'),(23,'E-GEOD-47395','Rett syndrome','D015518'),(24,'E-GEOD-46664','Wilms tumor','D009396'),(25,'E-GEOD-43631','','');
/*!40000 ALTER TABLE `disease` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `gene`
--

DROP TABLE IF EXISTS `gene`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `gene` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `Gene` varchar(50) DEFAULT NULL,
  `GeneMGI` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=63 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `gene`
--

LOCK TABLES `gene` WRITE;
/*!40000 ALTER TABLE `gene` DISABLE KEYS */;
INSERT INTO `gene` VALUES (32,'E-GEOD-61216','Prmt5','MGI:1351645'),(33,'E-GEOD-60968','Kat2b','MGI:1343094'),(34,'E-GEOD-60968','Kat2a','MGI:1343101'),(35,'E-GEOD-59215','Gata3','MGI:95663'),(36,'E-GEOD-58996','Htt','MGI:96067'),(37,'E-GEOD-58973','Rel','MGI:97897'),(38,'E-GEOD-58972','Rela','MGI:103290'),(39,'E-GEOD-58971','Rel','MGI:97897'),(40,'E-GEOD-58905','Rr17','MGI:5313387'),(41,'E-GEOD-58498','Dgcr8','MGI:2151114'),(42,'E-GEOD-57959','Cryaa','MGI:88515'),(43,'E-GEOD-57837','Nfkbiz','MGI:1931595'),(44,'E-GEOD-57396','Trim24','MGI:109275'),(45,'E-GEOD-57147','Kmt2d','MGI:2682319'),(46,'E-GEOD-56127','Gtpbp2','MGI:1860138'),(47,'E-GEOD-56018','Rorc','MGI:104856'),(48,'E-GEOD-55377','Dicer1','MGI:2177178'),(49,'E-GEOD-55376','Dicer1','MGI:2177178'),(50,'E-GEOD-54631','Ern1','MGI:1930134'),(51,'E-GEOD-54154','Lepr','MGI:104993'),(52,'E-GEOD-51008','Myc','MGI:97250'),(53,'E-GEOD-49949','Zeb2','MGI:1344407'),(54,'E-GEOD-49555','Setd7','MGI:1920501'),(55,'E-GEOD-49555','Akt1s1','MGI:1914855'),(56,'E-GEOD-47395','STC1','MGI:109131'),(57,'E-GEOD-46664','Wt1','MGI:98968'),(58,'E-GEOD-43631','Rb1','MGI:97874'),(59,'E-GEOD-43631','Rbl2','MGI:105085'),(60,'E-GEOD-43631','Rbl1','MGI:103300');
/*!40000 ALTER TABLE `gene` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `genotype`
--

DROP TABLE IF EXISTS `genotype`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `genotype` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `Genotype` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=64 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `genotype`
--

LOCK TABLES `genotype` WRITE;
/*!40000 ALTER TABLE `genotype` DISABLE KEYS */;
INSERT INTO `genotype` VALUES (1,'E-GEOD-61216','Prmt5-/-'),(2,'E-GEOD-61216','Prmt5+/-'),(3,'E-GEOD-60968','Kat2b-/-,Kat2a+/-'),(4,'E-GEOD-60968   ','Kat2b-/-,Kat2a-/-'),(5,'E-GEOD-59215','Gata3f/f'),(6,'E-GEOD-59215','WT'),(7,'E-GEOD-58996','Htt-/-'),(8,'E-GEOD-58996','Tg(HDexon1)62Gpb-/-'),(9,'E-GEOD-58996','WT'),(10,'E-GEOD-58973','Rel-/-'),(11,'E-GEOD-58973','Rel+/+'),(12,'E-GEOD-58972','Rela-/-'),(13,'E-GEOD-58972','WT'),(14,'E-GEOD-58971','Rel-/-'),(15,'E-GEOD-58971','WT'),(16,'E-GEOD-58905','Rr17'),(17,'E-GEOD-58905','WT'),(18,'E-GEOD-58498','Dgcr8-/-'),(19,'E-GEOD-58498','WT'),(20,'E-GEOD-57959','Cryaa-/-'),(21,'E-GEOD-57959','WT'),(22,'E-GEOD-57837','Nfkbiz-/-'),(23,'E-GEOD-57837','WT'),(24,'E-GEOD-57396','Trim24(-/-)'),(25,'E-GEOD-57396','WT'),(26,'E-GEOD-57147','Kmt2d(-/-)'),(27,'E-GEOD-57147','WT'),(28,'E-GEOD-56127','Gtpbp2-/-; n-Tr20J/J'),(29,'E-GEOD-56127','Gtpbp2+/+,n-Tr20J/J'),(30,'E-GEOD-56127','Gtpbp2-/-; n-Tr20N/N'),(31,'E-GEOD-56127','Gtpbp2+/+,n-Tr20N/N'),(32,'E-GEOD-56018','Rorc-/-'),(33,'E-GEOD-56018','WT'),(34,'E-GEOD-55377','Dicer1-/-'),(35,'E-GEOD-55377','WT'),(36,'E-GEOD-55376','Dicer1-/-,iCre75−'),(37,'E-GEOD-55376','Dicer1+/+,iCre75+'),(38,'E-GEOD-54631','Ern1+/-'),(39,'E-GEOD-54631','Ern1-/-'),(40,'E-GEOD-54154','Lepr(db/db)'),(41,'E-GEOD-54154','Lepr(db/-)'),(42,'E-GEOD-51008','Myc-/-'),(43,'E-GEOD-51008','WT'),(44,'E-GEOD-49949','Zeb2-/-'),(45,'E-GEOD-49949','WT'),(46,'E-GEOD-49555','Setd7-/-,Akt1s1-/-'),(47,'E-GEOD-49555','WT'),(48,'E-GEOD-47395','STC1-/-'),(49,'E-GEOD-47395','WT'),(50,'E-GEOD-46664','Wt1-/flox;Cre-ERTM'),(51,'E-GEOD-46664','Wt1flox/flox'),(52,'E-GEOD-43631','Rb1f/fRbl2f/f,Rbl1-/-'),(53,'E-GEOD-43631','Rb1f/fRbl2f/f,Rbl1-/-;AlbCreER/+');
/*!40000 ALTER TABLE `genotype` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `inquery`
--

DROP TABLE IF EXISTS `inquery`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `inquery` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(50) NOT NULL,
  `name` varchar(50) NOT NULL,
  `email` varchar(50) NOT NULL,
  `PubMed` varchar(50) DEFAULT NULL,
  `comments` text NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `inquery`
--

LOCK TABLES `inquery` WRITE;
/*!40000 ALTER TABLE `inquery` DISABLE KEYS */;
/*!40000 ALTER TABLE `inquery` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `main`
--

DROP TABLE IF EXISTS `main`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `main` (
  `ArrayExpress` varchar(20) NOT NULL,
  `GEO` varchar(20) DEFAULT NULL,
  `Title` text,
  `description` text,
  `OtherFactors` varchar(50) DEFAULT NULL,
  `PI` varchar(50) DEFAULT NULL,
  `email` varchar(50) DEFAULT NULL,
  `Website` varchar(511) DEFAULT NULL,
  `GeoArea` varchar(50) DEFAULT NULL,
  `ResearchArea` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`ArrayExpress`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `main`
--

LOCK TABLES `main` WRITE;
/*!40000 ALTER TABLE `main` DISABLE KEYS */;
INSERT INTO `main` VALUES ('E-GEOD-43631','GSE43631','next generation sequencing facilitates quantitative analysis of control and TKO liver transcriptomes','We used RNAseq to analyse the transcriptomes of cycling and quiescent livers from cTKO (Rblox/lox;p130lox/lox;p107–/–) and control mice. mRNA profiles of cycling and quiescent control and TKO liver tissue were generated by deep sequencing in duplicate','aging','Julien Sage','julsage@stanford.edu','https://med.stanford.edu/profiles/julien-sage','USA','Hematogy and Oncology'),('E-GEOD-46664','GSE46664','Digital Gene Expression Tag Profiling after Wt1 deletion in Sertoli cell','The Wilms tumor gene, Wt1, is specifically expressed in Sertoli cells (SCs) which support spermatogenesis. Germ cell loss was found in Wt1 knockout mice. In  vitro  studies  demonstrated  that  Wt1  was  essential  for  cell  polarity maintenance  in  SCs. We used Digital Gene Expression Tag Profiling to detail the global programme of gene expression after Wt1 deletion in sertoli cell and identified 710 gene expression level change more than 2 times. Digital Gene Expression Tag Profiling  analysis  were  performed  using  mRNA  from  control  and Wt1-deficient   Primary Sertoli cells, every group primare sertoli cell were from more than 30 mice.','NA','Xiaona Wang','xiaonayn@126.com','http://english.rpb.ioz.cas.cn/groups/gaofei/','China','Sex determination and early gonad development'),('E-GEOD-47395','GSE47395','RNA Seq analysis of Mefs Isolated from STC1+/+ and STC1-/- mice','Evaluation of X-linked gene expression in MEFs isolated from STC1 wild type and knock out mice X chromosome inactivation (XCI) is initiated in cis by the Xist RNA, which coats the inactive X chromosome (Xi) from which it is produced.  We performed a large-scale RNA interference screen to identify trans-acting XCI factors (XCIFs) that comprise regulators of cell signaling and transcription. We find that the XCIFs promote Xist expression and/or localization to the Xi. One of the XCIFs, STC1, is a glycoprotein found in both the cytoplasm and nucleus and whose function is poorly understood. A homozygous mouse knockout of STC1 has a defect in XCI but surprisingly is phenotypically normal. We performed transcriptome profiling (RNA-Seq) experiments to determine whether the expression levels of X-encoded genes were elevated in Stc1-/- female MEFs. In these experiments, RNA was prepared from three independent cultures of Stc1+/+ or Stc1-/- female MEFs. RNA samples were processed and amplified followed by deep sequencing. The similarity of X-linked gene expression between Stc1+/+ and Stc1-/- MEFs was statistically significant. The vast majority of autosomal genes were also expressed at comparable levels in Stc1+/+ and Stc1-/- MEFs. Sequenced mRNA isolated from the STC1+/+ or STC1-/- MEFs.','aging','Michael Richard Green','Michael.Green@umassmed.edu','http://profiles.umassmed.edu/Profiles/display/132099','USA','Molecular, Cell and Cancer Biology'),('E-GEOD-49555','GSE49555','PR-SET7 inactivation causes hepatocyte necrosis and spontaneous development of hepatocellular carcinoma derived from ductal progenitor cells','PR-SET7-mediated histone-4 lysine-20 methylation has been implicated in mitotic condensation, DNA damage response and replication licencing. Here we show that PR-SET7 function in the liver is pivotal for maintaining genome integrity. Hepatocyte-specific deletion of PR-SET7 in mouse embryos resulted in G2 arrest followed by massive cell death and defect in liver organogenesis. Inactivation at postnatal stages caused cell duplication-dependent hepatocyte necrosis with unusual features of autophagy, termed \"endonucleosis\". Necrotic death was accompanied by inflammation, fibrosis and compensatory growth induction of neighboring hepatocytes and resident ductal progenitor cells. Prolonged necrotic-regenerative cycles coupled with oncogenic STAT3 activation replaced pre-existing hepatocytes with hepatocellular carcinoma derived entirely from ductal progenitor cells. Hepatocellular carcinoma in these mice displays a cancer stem cell gene signature specified by the co-expression of ductal progenitor markers and oncofetal genes. Mice carrying hepatocyte specific inactivation of PR-SET7 were generated in order to investigate the function of PR-SET7 histone methyl transferase in liver organogenesis, hepatocyte proliferation and liver regeneration. P15 WT mice were injected intra-peritoneally (ip) with 25ml per kg DEN (diethyl nitrosamine). Mice were examined for RNA expression at 8 months old.','DEN (diethyl nitrosamine) intra-peritoneal injecti','Iannis Talianidis','talianidis@fleming.gr','http://www.fleming.gr/research/molecular-biology/talianidis-lab','Greece','transcriptional regulation of the liver'),('E-GEOD-49949','GSe49949','Loss of Sip1 leads to migration defects and retention of ectodermal markers during lens development','We report the application of  RNA-sequencing technology for the high-throughput profiling of mammalian lens gene expression at embryonic day 15.5.  The lens has a particularly biased transcriptome, with the top 50 genes encoding approximately 90% of the protein content.  Using RNA-Seq, we have shown that there are over 7,700 genes being expressed in the lens at embryonic day 15.5.  As expected, the crystallins and many structural genes were amoung the most highly expressed; however, numerous genes expressed at lower transcript abundance were also identified.  This study provides a framework for the application of RNA-Seq technology towards characterization of the mammalian lens transcriptome during development. Using high-throughput RNA-sequencing, gene expression in the mammalian lens was compared between an inbred C56Bl/6<har> strain and a mix background strain at E15.5.  This analysis identifies almost 2,000 genes being differentially expressed between the inbred and mixed background lenses, ranging from 6.5 fold upregulated to 5.2 fold downregulated in the mixed background compared to the inbred strain.  This list does not include unknown/predicted genes or pseudogenes which are known to change between strains.  Further, it appears that approximately 98% of these genes are altered at levels less than 2.5 fold.  This study therefore provides a fold change threshold cutoff (2.5 fold) to use in the analysis of differentially expressed lens genes at E15.5 using RNA-Seq technology as it takes into account genetic variation due to background strain differences. SIP1 encodes a DNA-binding transcription factor that regulates multiple developmental processes as highlighted by the pleiotropic defects observed in Mowat-Wilson Syndrome, which results from mutations in this gene.  Further, in adults, dysregulated SIP1 expression has been implicated in both cancer and fibrotic diseases where it functionally links TGFb signaling to the loss of epithelial preferred gene expression.  In the ocular lens, an epithelial tissue important for vision, Sip1 is co-expressed with epithelial markers such as E-cadherin, and is required for the complete separation of the lens vesicle from the head ectoderm during early ocular morphogenesis.  However, the function of Sip1 after early lens morphogenesis is still unknown.   Here, we conditionally deleted Sip1 from the developing mouse lens shortly after lens vesicle closure, leading to defects in coordinated fiber cell tip migration, defective suture formation and cataract.  Interestingly, RNA-Sequencing analysis on Sip1 knockout lenses identified 190 differentially expressed genes, all of which are distinct from previously described Sip1 target genes involved in EMT/cancer.  Furthermore, 34% of the upregulated genes in the Sip1 knockout lenses are normally downregulated as the lens transitions from the lens vesicle to early lens, while 49% of the genes downregulated in the Sip1 knockout lenses are normally upregulated during early lens development.  Overall, these data imply that Sip1 plays a major role in reprogramming the lens vesicle away from a surface ectoderm cell fate towards that necessary for the development of a transparent lens and demonstrate that Sip1 regulates distinctly different sets of genes in different cellular contexts. RNA-Seq of inbred background wild type lenses at E15.5 RNA-Seq comparison of mixed background wild type controls and inbred wild type (C57Bl/6<har>) lenses at E15.5 RNA-Seq comparison of mixed background wild type controls and Sip1 conditional knockout lenses at E15.5','aging','Melinda K. Duncan','duncanm@udel.edu','http://www.bio.udel.edu/users/rduncan','USA','osteogenics'),('E-GEOD-51008','GSE51008','Selective transcriptional regulation by Myc in cellular growth control and lymphomagenesis [RNA-Seq]','The c-myc proto-oncogene product, Myc, is a transcription factor that binds thousands of genomic loci. Recent work suggested that rather than up- and down-regulating selected groups of genes, Myc targets all active promoters and enhancers in the genome (a phenomenon termed \"invasion\") and acts as a general amplifier of transcription. However, the available data did not readily discriminate between direct and indirect effects of Myc on RNA biogenesis. We addressed this issue with genome-wide chromatin immunoprecipitation and RNA expression profiles during B-cell lymphomagenesis in mice, in cultured B-cells and fibroblasts. Consistent with long-standing observations, we detected general increases in total RNA or mRNA copies per cell (hereby termed \"amplification\") when comparing actively proliferating cells with control quiescent cells: this was true whether cells were stimulated by mitogens (requiring endogenous Myc for a proliferative response) or by deregulated, oncogenic Myc activity. RNA amplification and promoter/enhancer invasion by Myc were separable phenomena that could occur without one another. Moreover, whether or not associated with RNA amplification, Myc drove the differential expression of distinct subsets of target genes. Hence, while having the potential to interact with all active/poised regulatory elements in the genome, Myc does not directly act as a global transcriptional amplifier. Instead, our results imply that Myc activates and represses transcription of discrete gene sets, leading to changes in cellular state that can in turn feed back on global RNA production and turnover. Total RNA profiling of gene expression upon Myc regulation in two different cell types by Illumina sequencing','aging','Bruno Amati','bruno.amati@ifom-ieo-campus.it','http://genomics.iit.it/groups/oncogenes-chromatin-and-gene-regulation.html','Italy','Oncogenes, Chromatin and Gene Regulation'),('E-GEOD-54154','GSE54154','RNA-seq analysis of diabetes induced changes in macrophage transcriptome','Macrophage dysfunction and polarization plays key role in chronic inflammation associated with diabetes and its complications. However, the effect of diabetes on macrophage transcriptome including long non-coding RNAs is not known. Here, we analyzed global changes in transcriptome of bone marrow macrophages isolated from type 2 diabetic db/db mice and control littermates db/+ mice using high throughput RNA-seq technique. Data analysis showed that expression of genes relevant to fibrosis, cell adhesion and inflammation were altered in diabetic db/db mice relative to control db/+ mice. Furthermore, expression of several known and novel long non coding RNAs and nearby genes was altered in db/db mice. Gene ontology and IPA showed activation of signaling netwroks relevant to  fibrosis, cell adhesion and inflammatory pathways . This study for the first time demonstrated that diabetes profoundly affects macrophage transcriptome including expression of long non coding RNAs and altered the levels of genes relevant to diabetes complications. Bone marrow macrophages were isolated from 12 weeks old type 2 diabetic male db/db mice and control littermates db/+ mice. These were differentiated in culture for 7-8 days in the presence of 10  ng/ml of MCSF-1  (BMMC) or 20 ng/ml of GM-CSF (BMGM). Then RNA was extracted and used for RNA-seq.','MCSF-1 (BMMC) or GM-CSF (BMGM) treatment','Nancy Chen','zhuchen@coh.org','http://www.cityofhope.org/division-of-molecular-diabetes-research','USA','molecular diabetes'),('E-GEOD-54631','GSE54631','Novel Bioinformatics Method for Identification of Genome-Wide Non-Canonical Spliced Regions using RNA-Seq Data','We developed a bioinformatics approach called the Read-Split-Walk (RSW) pipeline, and evaluated it using two Ire1α heterozygous and two Ire1α-null samples. The 26nt non-canonical splice site in Xbp1 was detected as the top hit by our RSW pipeline in heterozygous samples but not in the negative control Ire1α knockout samples. We compared the Xbp1 results from our approach with results using the alignment program BWA, STAR, Exonerate and the Unix “grep” command.  We then applied our RSW pipeline to RNA-Seq data from the SKBR3 human breast cancer cell line. RSW reported a large number of non-canonical spliced regions for 108 genes in chromosome 17, which were identified by an independent study. Identification of non-canonical spliced regions for mouse MEF samples (two Ire1α heterozygous and two Ire1α-null samples)','cytosol induced stress in the endoplasmic reticulu','Randal J. Kaufman','rkaufman@sanfordburnham.org','http://labs.sanfordburnham.org/centerandlabs/neuroagingstem/kaufmanlab/Pages/Home.aspx','USA','protein synthesis'),('E-GEOD-55376','GSE55376','small RNAseq analysis of the global retinal transcriptome of rod photoreceptor-specific Dicer1 conditional knockout mice and control littermates','We report RNAseq analysis of the transcriptome of retinas from mature rod-specific Dicer1 cKO mice and control littermates lacking Cre expression in order to better understand changes in gene regulation that could lead to retinal degeneration in cKO mice. Examine retinal transcriptome of 3 biological replicates for each genotype from 4-week-old animals with tissue collected between 8:00 - 10:00AM','aging','Krzysztof Palczewski','kxp65@case.edu','http://pharmacology.case.edu/department/faculty/Primary/Pages/Palczewski/default.aspx','USA','pharmacology'),('E-GEOD-55377','GSE55377','RNAseq analysis of the global retinal transcriptome of rod photoreceptor-specific Dicer1 conditional knockout mice and control littermates','We report RNAseq analysis of the transcriptome of retinas from mature rod-specific Dicer1 cKO mice and control littermates lacking Cre expression in order to better understand changes in gene regulation that could lead to retinal degeneration in cKO mice. Examine retinal transcriptome of 3 biological replicates for each genotype from 4-week-old animals with tissue collected between 8:00 - 10:00AM','aging','Krzysztof Palczewski','kxp65@case.edu','http://pharmacology.case.edu/department/faculty/Primary/Pages/Palczewski/default.aspx','USA','pharmacology'),('E-GEOD-56018','GSE56018','Chemical Inhibition of the RORγt-dependent Transcriptional Network in Th17 cells [RNA-Seq]','RORγt is a transcription factor required for T helper 17 (Th17) cell development. We identified three RORγt-specific inhibitors that suppress Th17 cell responses including Th17 cell-mediated autoimmune disease. We systemically characterized RORγt binding data in the presence and absence of drug with corresponding whole-transcriptome sequencing for wild-type and RORγt-deficient cells. RORγt is central in a densely interconnected regulatory network, acting both as a direct activator of genes important for Th17 cell differentiation and as a direct repressor of genes from other T-cell lineages. The three inhibitors identified here reversed both of these modes of action, but to varying extents and through distinct mechanisms. Whereas one inhibitor displaced RORγt from its target-loci, the two more potent inhibitors affected transcription predominantly without removing DNA-binding. Our work illustrates the power of a system-scale analysis of transcriptional regulation to characterize potential therapeutic compounds that inhibit pathogenic Th17 cells and suppress autoimmunity. Transcriptional profiling of Th17 cells under chemical perturbations of RORγt, DMSO, and knockout of  RORγt','TMP778, TMP920, digoxin, or DMSO treatment','Vijay Kuchroo','vkuchroo@rics.bwh.harvard.edu','http://kuchroo-lab.bwh.harvard.edu','USA','Regulation of Autoimmune T Cell Responses'),('E-GEOD-56127','GSE56127','Ribosome stalling induced by mutation of a CNS-specific tRNA causes neurodegeneration','In higher eukaryotes, the large numbers of nuclear-encoded tRNA genes partially ensure the robustness of cytoplasmic protein translation. Here we discover that a loss-of-function in n-Tr20, a member of the nuclear-encoded tRNA Arg UCU family that is expressed specifically in the central nervous systems leads to low but detectable levels of ribosome stalling. In the absence of GTPBP2, a novel binding partner of the ribosome recycling protein Pelota, ribosome stalling increases, leading to widespread neurodegeneration. Our results not only define GTPBP2 as a ribosome rescue factor, but also unmask the disease potential of mutations in nuclear-encoded tRNA genes. In this submission we provide ribosome footprinting data from the cerebella of four strains derived from the C57BL/6J strain with combinations of  n-Tr20 and GTPBP2 mutations. Examination of ribosome stalling in cerebella from 4 mouse strains derived from the: C57BL/6J (B6J) strain. The nmf205-/- strain has a homozygous mutation in the gene GTPBP2 while the B6J strain has normal GTPBP2. The n-Tr20 J/J strain has a defect in the n-Tr20 tRNA while the n-Tr20 N/N strain has a functional n-Tr20 tRNA. The 4 strains are the 2x2 combinations of these defects and correctly functioning sequences. 2 replicates for each strain. Please note that only BAM files are included in the records since they form the basis of the study\'s conclusions.  The raw data ribosomal RNA have been filtered and then unique reads mapping to mm10 were computed using tophat and igenome annotations.','aging','Susan L. Ackerman','susan.ackerman@jax.org','http://www.hhmi.org/scientists/susan-l-ackerman','USA','Genetics, Neuroscience'),('E-GEOD-57147','GSE57147','DNA-damage induced differentiation of leukemic cells as an anti-cancer barrier','Here we show that the histone methyltransferase MLL4 (Kmt2d) is required for stem cell activity and an aggressive form of acute myeloid leukemia (AML) harboring the MLL-AF9 oncogene. MLL4 exerts its function by regulating transcriptional programs associated with the anti-oxidant response. The role of Mll4 (Kmt2d) in regulating the transcriptome of primary and transformed hematopoietic stem cells was studied.','polyIC injections','Andre Nussenzweig','nussenza@mail.nih.gov','https://ccr.cancer.gov/andre-nussenzweig','USA','genomic Instability, DNA damage/repair, breast/ova'),('E-GEOD-57396','GSE57396','An aging-like phenotype occurs in Tif1γ-/- hematopoietic stem cells','To determine whether an accelerated aging-like phenotype occurs in hematopoiesis of young Tif1γ-/- mice (4 months old), we purified 200,000 hematopoietic stem cells (LSK: Lineage negative, Sca1+, c-Kit+) from Tif1γ-/- mice and performed high-throughput mRNA sequencing (RNA-seq). We compared this transcriptome to physiological aging by creating two other RNAseq libraries from young (4 months old) and old (20 months old) wild type mice. RNAseq study on young Tif1γ-/- mice (4 months old), young wild type mice (4 months old) and old wild type mice (20 months old).','aging','Ronan Quéré','ronan.quere@inserm.fr','http://inserm-u866.u-bourgogne.fr/en/02-them-en.php','France','Hematopoiesis & leukemogenesis gene regulation'),('E-GEOD-57837','GSE57837','Genome wide analysis of gene expression in LPS stimulated splenic B cell','Analysis of class switch recombination, maturation or differenciation of B cells at gene expression levels. The hypothesis tested in the present study was that TLR signaling in B cells plays an pivotal role for class switch, maturation and differenciation. Total RNA obtained from cultured splenic B cells. Gene expression compared between control and cKO B cells.','IκB-ζ deficiency','Takashi MaruYama','ta-maru@umin.ac.jp.','http://kinou.chem.tohoku.ac.jp/KobayashiLabHP/Home_English.html','Japan','molecular chemistry'),('E-GEOD-57959','GSE57959','Comparison of expression data of  2 & 4- months old  CRYAAN101D versus CRYAAWT lenses','To understand the global view of dysregulated genes and pathwyas in CRYAAN101D lenses,  RNA sequencing of 2 & 4  months old CRYAAWT and CRYAAN101D lenses was carried out. Determination of differential gene expression between CRYAAWT and CRYAAN101D in 2 & 4 months old lenses','aging','Om Srivastava','srivasta@uab.edu','http://www.uab.edu/optometry/home/faculty-staff/faculty-directory/142-srivastavaom','USA','vision research'),('E-GEOD-58498','GSE58498','Expression data of wild type and C-DGCR8 KO cones at different timepoints [RNA-seq]','We have analyzed gene expression in cone photoreceptors isolated from wild type and C-DGCR8 (DiGeorge Syndrome Critical Region Gene 8) KO mice at five different time points to get a mechanistic inside into the altered molecular pathways after microRNAs depletion. Cones were isolated by FACS from wild type and C-DGCR8 KO mice expressing Cre recombinase postnatally specifically in cones (D4-Cre) and bearing conditional null Dgcr8 allele. Experiments were performed in duplicates from different time points (P30, P40, P50, P60, and P90). Total RNA was extracted and next-generation RNA sequencing was performed.','aging','Witold Filipowicz','witold.filipowicz@fmi.ch','http://www.fmi.ch/about/people/emeriti/emeriti.html?group=9','Switzerland','epigenetics, mechanisms of cancer, nuerobiology'),('E-GEOD-58905','GSE58905','Gene expression profiling of wild type and CNS2-deficient Treg cells in the resting state','Resting regulatory T cells (CD4+GFP+CD44lowCD62Lhigh) were sorted from the littermates of male Foxp3gfp and Foxp3ΔCNS2-gfp mice. Poly(A)-tailed RNA were amplified and sequenced on an Ion Torrent platform. RNA-seq of WT and CNS2KO regulatory T cells in the resting state','aging','Alexander Y. Rudensky','rudenska@mskcc.org','http://www.hhmi.org/scientists/alexander-y-rudensky','USA','regulatory T cells'),('E-GEOD-58971','GSE58971','RNAseq data from in vitro-stimulated (6 hours) murine relfl/flCD19-Cre and CD19-Cre splenic B cells purified by magnetic cell separation (REL6)','RNAseq analysis of CD40 + IgM in vitro-stimulated (6 hours) murine relfl/flCD19-Cre (furtheron designated as REL) and CD19-Cre (furtheron designated as WT) splenic B cells identifies genes regulated by the transcription factor c-REL in activated B cells. Splenic B cells from 12-week old relfl/flCD19-Cre and CD19-Cre littermate mice were isolated by magnetic cell separation from splenic mononuclear cells and stimulated in vitro for 6 hours with anti-CD40 and anti-IgM. RNA was isolated and submitted for RNA-sequencing on an Illumina HiSeq2000 instrument for 30 million single-ended reads.','anti-CD40 and anti-IgM stimulus','Ulf Klein','uk30@columbia.edu','http://www.microbiology.columbia.edu/faculty/klein.html','USA','Microbiology & Immunology '),('E-GEOD-58972','GSE58972','RNAseq data from in vitro-stimulated (6 hours) murine relafl/flCD19-Cre and CD19-Cre splenic B cells purified by magnetic cell separation (RELA)','RNAseq analysis of CD40 + IgM in vitro-stimulated (6 hours) murine relafl/flCD19-Cre (furtheron designated as RELA) and CD19-Cre (furtheron designated as WT) splenic B cells identifies genes regulated by the transcription factor RELA in activated B cells. Splenic B cells from 12-week old relafl/flCD19-Cre and CD19-Cre littermate mice were isolated by magnetic cell separation from splenic mononuclear cells and stimulated in vitro for 6 hours with anti-CD40 and anti-IgM. RNA was isolated and submitted for RNA-sequencing on an Illumina HiSeq2000 instrument for 30 million single-ended reads.','anti-CD40 and anti-IgM stimulus','Ulf Klein','uk30@columbia.edu','http://www.microbiology.columbia.edu/faculty/klein.html','USA','Microbiology & Immunology '),('E-GEOD-58973','GSE58973','RNAseq data from in vitro-stimulated (24 hours) murine relfl/flCD19-Cre and CD19-Cre splenic B cells purified by magnetic cell separation','RNAseq analysis of CD40 + IgM in vitro-stimulated (24 hours) murine relfl/flCD19-Cre (furtheron designated as REL) and CD19-Cre (furtheron designated as WT) splenic B cells identifies genes regulated by the transcription factor c-REL in activated B cells. Splenic B cells from 12-week old relfl/flCD19-Cre and CD19-Cre littermate mice were isolated by magnetic cell separation from splenic mononuclear cells and stimulated in vitro for 24 hours with anti-CD40 and anti-IgM. RNA was isolated and submitted for RNA-sequencing on an Illumina HiSeq2000 instrument for 30 million single-ended reads.','anti-CD40 and anti-IgM stimulus','Ulf Klein','uk30@columbia.edu','http://www.microbiology.columbia.edu/faculty/klein.html','USA','Microbiology & Immunology '),('E-GEOD-58996','GSE58996','The CNS-Heart Axis is a Source of Cardiac Dysfunction in Mouse Models   of Huntington’s Disease','Purpose: Transcriptome profiling (RNA-seq) to microarray to evaluate transcriptional changes in the heart of HD mouse models Methods: Heart mRNA profiles of 4-weeks-old wild-type (WT) and R6/2 transgenic; 15-weeks-old WT and R6/2 transgenic mice; 8-month-old WT and HdhQ150 knock-in mice; 22-month-old WT and HdhQ150 knock-in mice were generated by deep sequencing, in triplicate, using Illumina Hi-seq 2000. Conclusions: Our study showed that there is no major transcriptional deregulation in the heart of mouse models of HD. Heart mRNA profiles of 4-weeks-old wild-type (WT) and R6/2 transgenic; 15-weeks-old WT and R6/2 transgenic mice; 8-month-old WT and HdhQ150 knock-in mice; 22-month-old WT and HdhQ150 knock-in mice were generated by deep sequencing, in triplicate, using Illumina Hi-seq 2000.','aging','Gillian P. Bates','gillian.bates@kcl.ac.uk','https://kclpure.kcl.ac.uk/portal/en/organisations/medical--molecular-genetics(1a64acc0-c473-49af-b7ad-d05f2eb7f367).html','UK','Medical and Molecular Genetics'),('E-GEOD-59215','GSE59215','Transcriptome impact of acute deletion of Gata3 in murine pro-T cells','The purpose of the study was to determine what genes in DN2 pro-T cells are immediately regulated by the transcription factor GATA-3, either as activation targets or as repression targets.  To do this, two pairs of Gata3-floxed and control pro-T cells were generated and analyzed by RNA-seq within the first day of deletion of the Gata3 gene. Pro-T cells were generated by differentiation in vitro on OP9-DL1 monolayers of fetal liver-derive precursors from wildtype or Gata3-floxed mice, and the Gata3 gene was acutely deleted by transduction with Cre retroviral vector.  Within 20 hr after transduction, samples of acutely Gata3-deleted and control DN2 cells were sorted and RNA prepared for RNA-seq analysis.   High-throughput sequencing of the samples was carried out.   Experimental Gata3 deleted samples in both cases were Gata3-floxed, ROSA26R-EYFP samples infected with Cre retrovirus and sorted for EYFP+ (Cre-activated) DN2 phenotype. Control for experiment 1: wildtype (C57BL/6) DN2 pro-T cells generated in parallel, also treated with Cre retrovirus but sorted only for DN2 phenotype.  Control for experiment 2: same genotype as experimental, but infected with a GFP+ empty retroviral vector and sorted for GFP+ DN2 phenotype. Two pairs of RNA-seq samples of DN2 pro-T cells were generated for comparison, each pair consisting of a Gata3-deleted sample plus a stage-matched control.','NA','Ellen Rothenberg','evroth@its.caltech.edu','http://www.bbe.caltech.edu/content/ellen-rothenberg','USA','Developmental Biology and Genetics, Microbiology a'),('E-GEOD-60968','GSE60968','Gcn5 and PCAF negatively regulate interferon β production through HAT-independent inhibition of TBK1 [RNA-Seq]','Gcn5/PCAF double knockout (dKO) leads to loss of the global H3K9ac. RNA-Seq was performed to define the changes of gene expression in response to Gcn5/PCAF deletion and H3K9ac loss PCAF-/-;Gcn5f/D MEFs were infected with retroviral Cre to delete Gcn5 to generate Gcn5/PCAF dKO cells, followed by RNA-Seq analysis using spike-in RNA as controls','NA','Kai Ge','kaig@niddk.nih.gov','http://irp.nih.gov/pi/kai-ge','USA','epigenetic regulation of PPAR γ and adipogenesis'),('E-GEOD-61216','GSE61216','PRMT5 protects genomic integrity during global DNA demethylation in primordial germ cells and preimplantation embryos','Primordial germ cells (PGCs) and preimplantation embryos undergo epigenetic reprogramming, which entails comprehensive erasure of DNA methylation. We found that PRMT5, an arginine methyltransferase, translocates from the cytoplasm to the nucleus during this process. Here we show that conditional loss of PRMT5 in early PGCs caused complete male and female sterility, which is preceded by upregulation of LINE1 and IAP transposons and DNA damage response. Similarly, loss of maternal-zygotic PRMT5 also leads to IAP upregulation and early embryonic lethality. We detected the PRMT5-catalysed repressive H2A/H4R3me2s modification on LINE1 and IAP in early wildtype PGCs, directly implicating this mark in the maintenance of transposon silencing during DNA hypomethylation. PRMT5 subsequently translocates back to the cytoplasm of PGCs to participate in the previously described PIWI-interacting RNA (piRNA) pathway that promotes transposon silencing via de novo DNA re-methylation. Thus, PRMT5 has a novel direct role in genome defense during preimplantation development and in PGCs at the time of global DNA demethylation PGCs mRNA profiles of embryonic day 11.5 dpc control (Blimp1Cre;Prmt5flox/+) and Prmt5 germ cell knockout (Blimp1Cre;Prmt5 flox/flox) were generated by deep sequencing (SOLiD next generation sequencing).','aging','Azim Surani','a.surani@gurdon.cam.ac.uk','http://www.gurdon.cam.ac.uk/research/surani','UK','Mammalian germ cells, pluripotency and epigenesis');
/*!40000 ALTER TABLE `main` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `publication`
--

DROP TABLE IF EXISTS `publication`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `publication` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `PubMed` varchar(50) DEFAULT NULL,
  `Title` text,
  `Abstract` text,
  `Journal` varchar(50) DEFAULT NULL,
  `Year` int(11) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=32 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `publication`
--

LOCK TABLES `publication` WRITE;
/*!40000 ALTER TABLE `publication` DISABLE KEYS */;
INSERT INTO `publication` VALUES (1,'E-GEOD-49949','24161570','Loss of Sip1 leads to migration defects and retention of ectodermal markers during lens development.','SIP1 encodes a DNA-binding transcription factor that regulates multiple developmental processes, as highlighted by the pleiotropic defects observed in Mowat-Wilson syndrome, which results from mutations in this gene. Further, in adults, dysregulated SIP1 expression has been implicated in both cancer and fibrotic diseases, where it functionally links TGFβ signaling to the loss of epithelial cell characteristics and gene expression. In the ocular lens, an epithelial tissue important for vision, Sip1 is co-expressed with epithelial markers, such as E-cadherin, and is required for the complete separation of the lens vesicle from the head ectoderm during early ocular morphogenesis. However, the function of Sip1 after early lens morphogenesis is still unknown. Here, we conditionally deleted Sip1 from the developing mouse lens shortly after lens vesicle closure, leading to defects in coordinated fiber cell tip migration, defective suture formation, and cataract. Interestingly, RNA-Sequencing analysis on Sip1 knockout lenses identified 190 differentially expressed genes, all of which are distinct from previously described Sip1 target genes. Furthermore, 34% of the genes with increased expression in the Sip1 knockout lenses are normally downregulated as the lens transitions from the lens vesicle to early lens, while 49% of the genes with decreased expression in the Sip1 knockout lenses are normally upregulated during early lens development. Overall, these data imply that Sip1 plays a major role in reprogramming the lens vesicle away from a surface ectoderm cell fate towards that necessary for the development of a transparent lens and demonstrate that Sip1 regulates distinctly different sets of genes in different cellular contexts.','Mech. Dev.',2013),(2,'E-GEOD-46664','24571983','Wt1 is involved in leydig cell steroid hormone biosynthesis by regulating paracrine factor expression in mice.','Wilms tumor gene, Wt1, is abundantly expressed in testis Sertoli cells. Our recent study demonstrated that Wt1 is involved in spermatogenesis by regulating Sertoli cell polarity. In the present study, we found that Wt1 is also required for steroidogenesis in Leydig cells and that deletion of the Wt1 gene resulted in defects in testosterone biosynthesis and downregulation of steroidogenic gene expression, including cytochrome P450 side-chain cleavage (P450scc), steroidogenic acute regulatory protein (StAR), 3beta-hydroxysteroid dehydrogenase I (3beta-HSD), and cytochrome P450 17A1 (Cyp17a1). The expression of LHR was significantly decreased in Wt1(-/flox); Cre-ER(TM) testes after tamoxifen induction, whereas the luteinizing hormone level in serum was unchanged. Further studies revealed that desert hedgehog (Dhh) expression was regulated by Wt1 in Sertoli cells and that its expression was significantly reduced in Wt1-deficient testes. In vitro study demonstrated that the defect in testosterone production and decreased expression of several steroidogenic genes in Wt1-deficient testis explants was partially rescued by smoothened agonist (SAG), a hedgehog pathway agonist. These results indicate that Wt1 is most likely involved in Leydig cell steroidogenesis by regulating the expression of paracrine factors in seminiferous tubules. Dhh probably had important roles in this process, but we could not exclude the possibility that other factors were also required for Leydig cell steroidogenesis. Loss of Wt1 leads to downregulation of paracrine factors, which in turn causes a decrease in steroidogenic enzyme expression and reduces testosterone production in Leydig cells. The results of this study further confirm that the cross talk between Sertoli cells and Leydig cells has important roles in Leydig cell steroidogenesis.','Biol. Reprod.',2014),(3,'E-GEOD-56018','24745332','Small-molecule RORγt antagonists inhibit T helper 17 cell transcriptional network by divergent mechanisms.','We identified three retinoid-related orphan receptor gamma t (RORγt)-specific inhibitors that suppress T helper 17 (Th17) cell responses, including Th17-cell-mediated autoimmune disease. We systemically characterized RORγt binding in the presence and absence of drugs with corresponding whole-genome transcriptome sequencing. RORγt acts as a direct activator of Th17 cell signature genes and a direct repressor of signature genes from other T cell lineages; its strongest transcriptional effects are on cis-regulatory sites containing the RORα binding motif. RORγt is central in a densely interconnected regulatory network that shapes the balance of T cell differentiation. Here, the three inhibitors modulated the RORγt-dependent transcriptional network to varying extents and through distinct mechanisms. Whereas one inhibitor displaced RORγt from its target loci, the other two inhibitors affected transcription predominantly without removing DNA binding. Our work illustrates the power of a system-scale analysis of transcriptional regulation to characterize potential therapeutic compounds that inhibit pathogenic Th17 cells and suppress autoimmunity.','Immunity',2014),(4,'E-GEOD-55376','24812086','DICER1 is essential for survival of postmitotic rod photoreceptor cells in mice.','Photoreceptor cell death is the proximal cause of blindness in many retinal degenerative disorders; hence, understanding the gene regulatory networks that promote photoreceptor survival is at the forefront of efforts to combat blindness. Down-regulation of the microRNA (miRNA)-processing enzyme DICER1 in the retinal pigmented epithelium has been implicated in geographic atrophy, an advanced form of age-related macular degeneration (AMD). However, little is known about the function of DICER1 in mature rod photoreceptor cells, another retinal cell type that is severely affected in AMD. Using a conditional-knockout (cKO) mouse model, we report that loss of DICER1 in mature postmitotic rods leads to robust retinal degeneration accompanied by loss of visual function. At 14 wk of age, cKO mice exhibit a 90% reduction in photoreceptor nuclei and a 97% reduction in visual chromophore compared with those in control littermates. Before degeneration, cKO mice do not exhibit significant defects in either phototransduction or the visual cycle, suggesting that miRNAs play a primary role in rod photoreceptor survival. Using comparative small RNA sequencing analysis, we identified rod photoreceptor miRNAs of the miR-22, miR-26, miR-30, miR-92, miR-124, and let-7 families as potential factors involved in regulating the survival of rods.','FASEB J.',2014),(5,'E-GEOD-55377','24812086','DICER1 is essential for survival of postmitotic rod photoreceptor cells in mice.','Photoreceptor cell death is the proximal cause of blindness in many retinal degenerative disorders; hence, understanding the gene regulatory networks that promote photoreceptor survival is at the forefront of efforts to combat blindness. Down-regulation of the microRNA (miRNA)-processing enzyme DICER1 in the retinal pigmented epithelium has been implicated in geographic atrophy, an advanced form of age-related macular degeneration (AMD). However, little is known about the function of DICER1 in mature rod photoreceptor cells, another retinal cell type that is severely affected in AMD. Using a conditional-knockout (cKO) mouse model, we report that loss of DICER1 in mature postmitotic rods leads to robust retinal degeneration accompanied by loss of visual function. At 14 wk of age, cKO mice exhibit a 90% reduction in photoreceptor nuclei and a 97% reduction in visual chromophore compared with those in control littermates. Before degeneration, cKO mice do not exhibit significant defects in either phototransduction or the visual cycle, suggesting that miRNAs play a primary role in rod photoreceptor survival. Using comparative small RNA sequencing analysis, we identified rod photoreceptor miRNAs of the miR-22, miR-26, miR-30, miR-92, miR-124, and let-7 families as potential factors involved in regulating the survival of rods.','FASEB J.',2014),(6,'E-GEOD-54631','24991935','Novel bioinformatics method for identification of genome-wide non-canonical spliced regions using RNA-Seq data.','SETTING: During endoplasmic reticulum (ER) stress, the endoribonuclease (RNase) Ire1α initiates removal of a 26 nt region from the mRNA encoding the transcription factor Xbp1 via an unconventional mechanism (atypically within the cytosol). This causes an open reading frame-shift that leads to altered transcriptional regulation of numerous downstream genes in response to ER stress as part of the unfolded protein response (UPR). Strikingly, other examples of targeted, unconventional splicing of short mRNA regions have yet to be reported.OBJECTIVE: Our goal was to develop an approach to identify non-canonical, possibly very short, splicing regions using RNA-Seq data and apply it to ER stress-induced Ire1α heterozygous and knockout mouse embryonic fibroblast (MEF) cell lines to identify additional Ire1α targets.RESULTS: We developed a bioinformatics approach called the Read-Split-Walk (RSW) pipeline, and evaluated it using two Ire1α heterozygous and two Ire1α-null samples. The 26 nt non-canonical splice site in Xbp1 was detected as the top hit by our RSW pipeline in heterozygous samples but not in the negative control Ire1α knockout samples. We compared the Xbp1 results from our approach with results using the alignment program BWA, Bowtie2, STAR, Exonerate and the Unix &quot;grep&quot; command. We then applied our RSW pipeline to RNA-Seq data from the SKBR3 human breast cancer cell line. RSW reported a large number of non-canonical spliced regions for 108 genes in chromosome 17, which were identified by an independent study.CONCLUSIONS: We conclude that our RSW pipeline is a practical approach for identifying non-canonical splice junction sites on a genome-wide level. We demonstrate that our pipeline can detect novel splice sites in RNA-Seq data generated under similar conditions for multiple species, in our case mouse and human.','PLoS ONE',2014),(7,'E-GEOD-58498','25002228','miRNAs 182 and 183 are necessary to maintain adult cone photoreceptor outer segments and visual function.','The outer segments of cones serve as light detectors for daylight color vision, and their dysfunction leads to human blindness conditions. We show that the cone-specific disruption of DGCR8 in adult mice led to the loss of miRNAs and the loss of outer segments, resulting in photoreceptors with significantly reduced light responses. However, the number of cones remained unchanged. The loss of the outer segments occurred gradually over 1 month, and during this time the genetic signature of cones decreased. Reexpression of the sensory-cell-specific miR-182 and miR-183 prevented outer segment loss. These miRNAs were also necessary and sufficient for the formation of inner segments, connecting cilia and short outer segments, as well as light responses in stem-cell-derived retinal cultures. Our results show that miR-182- and miR-183-regulated pathways are necessary for cone outer segment maintenance in vivo and functional outer segment formation in vitro.','Neuron',2014),(8,'E-GEOD-57396','25002492','Tif1γ regulates the TGF-β1 receptor and promotes physiological aging of hematopoietic stem cells.','The hematopoietic system declines with age. Myeloid-biased differentiation and increased incidence of myeloid malignancies feature aging of hematopoietic stem cells (HSCs), but the mechanisms involved remain uncertain. Here, we report that 4-mo-old mice deleted for transcription intermediary factor 1γ (Tif1γ) in HSCs developed an accelerated aging phenotype. To reinforce this result, we also show that Tif1γ is down-regulated in HSCs during aging in 20-mo-old wild-type mice. We established that Tif1γ controls TGF-β1 receptor (Tgfbr1) turnover. Compared with young HSCs, Tif1γ(-/-) and old HSCs are more sensitive to TGF-β signaling. Importantly, we identified two populations of HSCs specifically discriminated by Tgfbr1 expression level and provided evidence of the capture of myeloid-biased (Tgfbr1(hi)) and myeloid-lymphoid-balanced (Tgfbr1(lo)) HSCs. In conclusion, our data provide a new paradigm for Tif1γ in regulating the balance between lymphoid- and myeloid-derived HSCs through TGF-β signaling, leading to HSC aging.','Proc. Natl. Acad. Sci. U.S.A.',2014),(9,'E-GEOD-54154','25008173','Regulation of inflammatory phenotype in macrophages by a diabetes-induced long noncoding RNA.','The mechanisms by which macrophages mediate the enhanced inflammation associated with diabetes complications are not completely understood. We used RNA sequencing to profile the transcriptome of bone marrow macrophages isolated from diabetic db/db mice and identified 1,648 differentially expressed genes compared with control db/+ mice. Data analyses revealed that diabetes promoted a proinflammatory, profibrotic, and dysfunctional alternatively activated macrophage phenotype possibly via transcription factors involved in macrophage function. Notably, diabetes altered levels of several long noncoding RNAs (lncRNAs). Because the role of lncRNAs in diabetes complications is unknown, we further characterized the function of lncRNA E330013P06, which was upregulated in macrophages from db/db and diet-induced insulin-resistant type 2 diabetic (T2D) mice, but not from type 1 diabetic mice. It was also upregulated in monocytes from T2D patients. E330013P06 was also increased along with inflammatory genes in mouse macrophages treated with high glucose and palmitic acid. E330013P06 overexpression in macrophages induced inflammatory genes, enhanced responses to inflammatory signals, and increased foam cell formation. In contrast, small interfering RNA-mediated E330013P06 gene silencing inhibited inflammatory genes induced by the diabetic stimuli. These results define the diabetic macrophage transcriptome and novel functional roles for lncRNAs in macrophages that could lead to lncRNA-based therapies for inflammatory diabetes complications.','Diabetes',2014),(10,'E-GEOD-43631','25017070','Organ size control is dominant over Rb family inactivation to restrict proliferation in vivo.','In mammals, a cell\'s decision to divide is thought to be under the control of the Rb/E2F pathway. We previously found that inactivation of the Rb family of cell cycle inhibitors (Rb, p107, and p130) in quiescent liver progenitors leads to uncontrolled division and cancer initiation. Here, we show that, in contrast, deletion of the entire Rb gene family in mature hepatocytes is not sufficient for their long-term proliferation. The cell cycle block in Rb family mutant hepatocytes is independent of the Arf/p53/p21 checkpoint but can be abrogated upon decreasing liver size. At the molecular level, we identify YAP, a transcriptional regulator involved in organ size control, as a factor required for the sustained expression of cell cycle genes in hepatocytes. These experiments identify a higher level of regulation of the cell cycle in vivo in which signals regulating organ size are dominant regulators of the core cell cycle machinery.','Cell Rep',2014),(11,'E-GEOD-51008','25043028','Selective transcriptional regulation by Myc in cellular growth control and lymphomagenesis.','The c-myc proto-oncogene product, Myc, is a transcription factor that binds thousands of genomic loci. Recent work suggested that rather than up- and downregulating selected groups of genes, Myc targets all active promoters and enhancers in the genome (a phenomenon termed \'invasion\') and acts as a general amplifier of transcription. However, the available data did not readily discriminate between direct and indirect effects of Myc on RNA biogenesis. We addressed this issue with genome-wide chromatin immunoprecipitation and RNA expression profiles during B-cell lymphomagenesis in mice, in cultured B cells and fibroblasts. Consistent with long-standing observations, we detected general increases in total RNA or messenger RNA copies per cell (hereby termed \'amplification\') when comparing actively proliferating cells with control quiescent cells: this was true whether cells were stimulated by mitogens (requiring endogenous Myc for a proliferative response) or by deregulated, oncogenic Myc activity. RNA amplification and promoter/enhancer invasion by Myc were separable phenomena that could occur without one another. Moreover, whether or not associated with RNA amplification, Myc drove the differential expression of distinct subsets of target genes. Hence, although having the potential to interact with all active or poised regulatory elements in the genome, Myc does not directly act as a global transcriptional amplifier. Instead, our results indicate that Myc activates and represses transcription of discrete gene sets, leading to changes in cellular state that can in turn feed back on global RNA production and turnover.','Nature',2014),(12,'E-GEOD-56127','25061210','RNA function. Ribosome stalling induced by mutation of a CNS-specific tRNA causes neurodegeneration.','In higher eukaryotes, transfer RNAs (tRNAs) with the same anticodon are encoded by multiple nuclear genes, and little is known about how mutations in these genes affect translation and cellular homeostasis. Similarly, the surveillance systems that respond to such defects in higher eukaryotes are not clear. Here, we discover that loss of GTPBP2, a novel binding partner of the ribosome recycling protein Pelota, in mice with a mutation in a tRNA gene that is specifically expressed in the central nervous system causes ribosome stalling and widespread neurodegeneration. Our results not only define GTPBP2 as a ribosome rescue factor but also unmask the disease potential of mutations in nuclear-encoded tRNA genes.','Science',2014),(13,'E-GEOD-57147','25079327','DNA-damage-induced differentiation of leukaemic cells as an anti-cancer barrier.','Self-renewal is the hallmark feature both of normal stem cells and cancer stem cells. Since the regenerative capacity of normal haematopoietic stem cells is limited by the accumulation of reactive oxygen species and DNA double-strand breaks, we speculated that DNA damage might also constrain leukaemic self-renewal and malignant haematopoiesis. Here we show that the histone methyl-transferase MLL4, a suppressor of B-cell lymphoma, is required for stem-cell activity and an aggressive form of acute myeloid leukaemia harbouring the MLL-AF9 oncogene. Deletion of MLL4 enhances myelopoiesis and myeloid differentiation of leukaemic blasts, which protects mice from death related to acute myeloid leukaemia. MLL4 exerts its function by regulating transcriptional programs associated with the antioxidant response. Addition of reactive oxygen species scavengers or ectopic expression of FOXO3 protects MLL4(-/-) MLL-AF9 cells from DNA damage and inhibits myeloid maturation. Similar to MLL4 deficiency, loss of ATM or BRCA1 sensitizes transformed cells to differentiation, suggesting that myeloid differentiation is promoted by loss of genome integrity. Indeed, we show that restriction-enzyme-induced double-strand breaks are sufficient to induce differentiation of MLL-AF9 blasts, which requires cyclin-dependent kinase inhibitor p21(Cip1) (Cdkn1a) activity. In summary, we have uncovered an unexpected tumour-promoting role of genome guardians in enforcing the oncogene-induced differentiation blockade in acute myeloid leukaemia.','Nature',2014),(14,'E-GEOD-58996','25101683','Dysfunction of the CNS-heart axis in mouse models of Huntington\'s disease.','Cardiac remodelling and contractile dysfunction occur during both acute and chronic disease processes including the accumulation of insoluble aggregates of misfolded amyloid proteins that are typical features of Alzheimer\'s, Parkinson\'s and Huntington\'s disease (HD). While HD has been described mainly as a neurological disease, multiple epidemiological studies have shown that HD patients exhibit a high incidence of cardiovascular events leading to heart failure, and that this is the second highest cause of death. Given that huntingtin is ubiquitously expressed, cardiomyocytes may be at risk of an HD-related dysfunction. In mice, the forced expression of an expanded polyQ repeat under the control of a cardiac specific promoter led to severe heart failure followed by reduced lifespan. However the mechanism leading to cardiac dysfunction in the clinical and pre-clinical HD settings remains unknown. To unravel this mechanism, we employed the R6/2 transgenic and HdhQ150 knock-in mouse models of HD. We found that pre-symptomatic animals developed connexin-43 relocation and a significant deregulation of hypertrophic markers and Bdnf transcripts. In the symptomatic animals, pronounced functional changes were visualised by cardiac MRI revealing a contractile dysfunction, which might be a part of dilatated cardiomyopathy (DCM). This was accompanied by the re-expression of foetal genes, apoptotic cardiomyocyte loss and a moderate degree of interstitial fibrosis. To our surprise, we could identify neither mutant HTT aggregates in cardiac tissue nor a HD-specific transcriptional dysregulation, even at the end stage of disease. We postulate that the HD-related cardiomyopathy is caused by altered central autonomic pathways although the pathogenic effects of mutant HTT acting intrinsically in the heart may also be a contributing factor.','PLoS Genet.',2014),(15,'E-GEOD-57837','25124037','Control of Toll-like receptor-mediated T cell-independent type 1 antibody responses by the inducible nuclear protein IκB-ζ.','Antibody responses have been classified as being either T cell-dependent or T cell-independent (TI). TI antibody responses are further classified as being either type 1 (TI-1) or type 2 (TI-2), depending on their requirement for B cell-mediated antigen receptor signaling. Although the mechanistic basis of antibody responses has been studied extensively, it remains unclear whether different antibody responses share similarities in their transcriptional regulation. Here, we show that mice deficient in IκB-ζ, specifically in their B cells, have impaired TI-1 antibody responses but normal T cell-dependent and TI-2 antibody responses. The absence of IκB-ζ in B cells also impaired proliferation triggered by Toll-like receptor (TLR) activation, plasma cell differentiation, and class switch recombination (CSR). Mechanistically, IκB-ζ-deficient B cells could not induce TLR-mediated induction of activation-induced cytidine deaminase (AID), a class-switch DNA recombinase. Retroviral transduction of AID in IκB-ζ-deficient B cells restored CSR activity. Furthermore, acetylation of histone H3 in the vicinity of the transcription start site of the gene that encodes AID was reduced in IκB-ζ-deficient B cells relative to IκB-ζ-expressing B cells. These results indicate that IκB-ζ regulates TLR-mediated CSR by inducing AID. Moreover, IκB-ζ defines differences in the transcriptional regulation of different antibody responses.','J. Biol. Chem.',2014),(16,'E-GEOD-58905','25126783','Control of the inheritance of regulatory T cell identity by a cis element in the Foxp3 locus.','In multicellular organisms, specialized functions are delegated to distinct cell types whose identity and functional integrity are maintained upon challenge. However, little is known about the mechanisms enabling lineage inheritance and their biological implications. Regulatory T (Treg) cells, which express the transcription factor Foxp3, suppress fatal autoimmunity throughout the lifespan of animals. Here, we show that a dedicated Foxp3 intronic element CNS2 maintains Treg cell lineage identity by acting as a sensor of the essential Treg cell growth factor IL-2 and its downstream target STAT5. CNS2 sustains Foxp3 expression during division of mature Treg cells when IL-2 is limiting and counteracts proinflammatory cytokine signaling that leads to the loss of Foxp3. CNS2-mediated stable inheritance of Foxp3 expression is critical for adequate suppression of diverse types of chronic inflammation by Treg cells and prevents their differentiation into inflammatory effector cells. The described mechanism may represent a general principle of the inheritance of differentiated cell states.','Cell',2014),(17,'E-GEOD-47395','25136103','Genetic and pharmacological reactivation of the mammalian inactive X chromosome.','X-chromosome inactivation (XCI), the random transcriptional silencing of one X chromosome in somatic cells of female mammals, is a mechanism that ensures equal expression of X-linked genes in both sexes. XCI is initiated in cis by the noncoding Xist RNA, which coats the inactive X chromosome (Xi) from which it is produced. However, trans-acting factors that mediate XCI remain largely unknown. Here, we perform a large-scale RNA interference screen to identify trans-acting XCI factors (XCIFs) that comprise regulators of cell signaling and transcription, including the DNA methyltransferase, DNMT1. The expression pattern of the XCIFs explains the selective onset of XCI following differentiation. The XCIFs function, at least in part, by promoting expression and/or localization of Xist to the Xi. Surprisingly, we find that DNMT1, which is generally a transcriptional repressor, is an activator of Xist transcription. Small-molecule inhibitors of two of the XCIFs can reversibly reactivate the Xi, which has implications for treatment of Rett syndrome and other dominant X-linked diseases. A homozygous mouse knockout of one of the XCIFs, stanniocalcin 1 (STC1), has an expected XCI defect but surprisingly is phenotypically normal. Remarkably, X-linked genes are not overexpressed in female Stc1(-/-) mice, revealing the existence of a mechanism(s) that can compensate for a persistent XCI deficiency to regulate X-linked gene expression.','Proc. Natl. Acad. Sci. U.S.A.',2014),(18,'E-GEOD-57959','25146988','Molecular mechanism of formation of cortical opacity in CRYAAN101D transgenic mice.','PURPOSE: The CRYAAN101D transgenic mouse model expressing deamidated αA-crystallin (deamidation at N101 position to D) develops cortical cataract at the age of 7 to 9 months. The present study was carried out to explore the molecular mechanism that leads to the development of cortical opacity in CRYAAN101D lenses.METHODS: RNA sequence analysis was carried out on 2- and 4-month-old αA-N101D and wild type (WT) lenses. To understand the biologic relevance and function of significantly altered genes, Ingenuity Pathway Analysis (IPA) was done. To elucidate terminal differentiation defects, immunohistochemical, and Western blot analyses were carried out.RESULTS: RNA sequence and IPA data suggested that the genes belonging to gene expression, cellular assembly and organization, and cell cycle and apoptosis networks were altered in N101D lenses. In addition, the tight junction signaling and Rho A signaling were among the top three canonical pathways that were affected in N101D mutant. Immunohistochemical analysis identified a series of terminal differentiation defects in N101D lenses, specifically, increased proliferation and decreased differentiation of lens epithelial cells (LEC) and decreased denucleation of lens fiber cells (LFC). The expression of Rho A was reduced in different-aged N101D lenses, and, conversely, Cdc42 and Rac1 expressions were increased in the N101D mutants. Moreover, earlier in development, the expression of major membrane-bound molecular transporter Na,K-ATPase was drastically reduced in N101D lenses.CONCLUSIONS: The results suggest that the terminal differentiation defects, specifically, increased proliferation and decreased denucleation are responsible for the development of lens opacity in N101D lenses.','Invest. Ophthalmol. Vis. Sci.',2014),(19,'E-GEOD-59215','25172496','GATA-3 dose-dependent checkpoints in early T cell commitment.','GATA-3 expression is crucial for T cell development and peaks during commitment to the T cell lineage, midway through the CD4(-)CD8(-) (double-negative [DN]) stages 1-3. We used RNA interference and conditional deletion to reduce GATA-3 protein acutely at specific points during T cell differentiation in vitro. Even moderate GATA-3 reduction killed DN1 cells, delayed progression to the DN2 stage, skewed DN2 gene regulation, and blocked appearance of the DN3 phenotype. Although a Bcl-2 transgene rescued DN1 survival and improved DN2 cell generation, it did not restore DN3 differentiation. Gene expression analyses (quantitative PCR, RNA sequencing) showed that GATA-3-deficient DN2 cells quickly upregulated genes, including Spi1 (PU.1) and Bcl11a, and downregulated genes, including Cpa3, Ets1, Zfpm1, Bcl11b, Il9r, and Il17rb with gene-specific kinetics and dose dependencies. These targets could mediate two distinct roles played by GATA-3 in lineage commitment, as revealed by removing wild-type or GATA-3-deficient early T lineage cells from environmental Notch signals. GATA-3 worked as a potent repressor of B cell potential even at low expression levels, so that only full deletion of GATA-3 enabled pro-T cells to reveal B cell potential. The ability of GATA-3 to block B cell development did not require T lineage commitment factor Bcl11b. In prethymic multipotent precursors, however, titration of GATA-3 activity using tamoxifen-inducible GATA-3 showed that GATA-3 inhibits B and myeloid developmental alternatives at different threshold doses. Furthermore, differential impacts of a GATA-3 obligate repressor construct imply that B and myeloid development are inhibited through distinct transcriptional mechanisms. Thus, the pattern of GATA-3 expression sequentially produces B lineage exclusion, T lineage progression, and myeloid-lineage exclusion for commitment.','J. Immunol.',2014),(20,'E-GEOD-58971','25180063','Germinal center B cell maintenance and differentiation are controlled by distinct NF-κB transcription factor subunits.','Germinal centers (GCs) are the sites where memory B cells and plasma cells producing high-affinity antibodies are generated during T cell-dependent immune responses. The molecular control of GC B cell maintenance and differentiation remains incompletely understood. Activation of the NF-κB signaling pathway has been implicated; however, the distinct roles of the individual NF-κB transcription factor subunits are unknown. We report that GC B cell-specific deletion of the NF-κB subunits c-REL or RELA, which are both activated by the canonical NF-κB pathway, abolished the generation of high-affinity B cells via different mechanisms acting at distinct stages during the GC reaction. c-REL deficiency led to the collapse of established GCs immediately after the formation of dark and light zones at day 7 of the GC reaction and was associated with the failure to activate a metabolic program that promotes cell growth. Conversely, RELA was dispensable for GC maintenance but essential for the development of GC-derived plasma cells due to impaired up-regulation of BLIMP1. These results indicate that activation of the canonical NF-κB pathway in GC B cells controls GC maintenance and differentiation through distinct transcription factor subunits. Our findings have implications for the role of NF-κB in GC lymphomagenesis.','J. Exp. Med.',2014),(21,'E-GEOD-58972','25180063','Germinal center B cell maintenance and differentiation are controlled by distinct NF-κB transcription factor subunits.','Germinal centers (GCs) are the sites where memory B cells and plasma cells producing high-affinity antibodies are generated during T cell-dependent immune responses. The molecular control of GC B cell maintenance and differentiation remains incompletely understood. Activation of the NF-κB signaling pathway has been implicated; however, the distinct roles of the individual NF-κB transcription factor subunits are unknown. We report that GC B cell-specific deletion of the NF-κB subunits c-REL or RELA, which are both activated by the canonical NF-κB pathway, abolished the generation of high-affinity B cells via different mechanisms acting at distinct stages during the GC reaction. c-REL deficiency led to the collapse of established GCs immediately after the formation of dark and light zones at day 7 of the GC reaction and was associated with the failure to activate a metabolic program that promotes cell growth. Conversely, RELA was dispensable for GC maintenance but essential for the development of GC-derived plasma cells due to impaired up-regulation of BLIMP1. These results indicate that activation of the canonical NF-κB pathway in GC B cells controls GC maintenance and differentiation through distinct transcription factor subunits. Our findings have implications for the role of NF-κB in GC lymphomagenesis.','J. Exp. Med.',2014),(22,'E-GEOD-58973','25180063','Germinal center B cell maintenance and differentiation are controlled by distinct NF-κB transcription factor subunits.','Germinal centers (GCs) are the sites where memory B cells and plasma cells producing high-affinity antibodies are generated during T cell-dependent immune responses. The molecular control of GC B cell maintenance and differentiation remains incompletely understood. Activation of the NF-κB signaling pathway has been implicated; however, the distinct roles of the individual NF-κB transcription factor subunits are unknown. We report that GC B cell-specific deletion of the NF-κB subunits c-REL or RELA, which are both activated by the canonical NF-κB pathway, abolished the generation of high-affinity B cells via different mechanisms acting at distinct stages during the GC reaction. c-REL deficiency led to the collapse of established GCs immediately after the formation of dark and light zones at day 7 of the GC reaction and was associated with the failure to activate a metabolic program that promotes cell growth. Conversely, RELA was dispensable for GC maintenance but essential for the development of GC-derived plasma cells due to impaired up-regulation of BLIMP1. These results indicate that activation of the canonical NF-κB pathway in GC B cells controls GC maintenance and differentiation through distinct transcription factor subunits. Our findings have implications for the role of NF-κB in GC lymphomagenesis.','J. Exp. Med.',2014),(23,'E-GEOD-60968','25269644','Gcn5 and PCAF negatively regulate interferon-β production through HAT-independent inhibition of TBK1.','Viral infection triggers innate immune signaling, which in turn induces interferon-β (IFN-β) production to establish innate antiviral immunity. Previous studies showed that Gcn5 (Kat2a), a histone acetyltransferase (HAT) with partial functional redundancy with PCAF (Kat2b), and Gcn5/PCAF-mediated histone H3K9 acetylation (H3K9ac) are enriched on the active IFNB gene promoter. However, whether Gcn5/PCAF and H3K9ac regulate IFN-β production is unknown. Here, we show that Gcn5/PCAF-mediated H3K9ac correlates well with, but is surprisingly dispensable for, the expression of endogenous IFNB and the vast majority of active genes in fibroblasts. Instead, Gcn5/PCAF repress IFN-β production and innate antiviral immunity in several cell types in a HAT-independent and non-transcriptional manner: by inhibiting the innate immune signaling kinase TBK1 in the cytoplasm. Our results thus identify Gcn5 and PCAF as negative regulators of IFN-β production and innate immune signaling.','EMBO Rep.',2014),(24,'E-GEOD-61216','25457166','PRMT5 protects genomic integrity during global DNA demethylation in primordial germ cells and preimplantation embryos.','Primordial germ cells (PGCs) and preimplantation embryos undergo epigenetic reprogramming, which includes comprehensive DNA demethylation. We found that PRMT5, an arginine methyltransferase, translocates from the cytoplasm to the nucleus during this process. Here we show that conditional loss of PRMT5 in early PGCs causes complete male and female sterility, preceded by the upregulation of LINE1 and IAP transposons as well as activation of a DNA damage response. Similarly, loss of maternal-zygotic PRMT5 also leads to IAP upregulation. PRMT5 is necessary for the repressive H2A/H4R3me2s chromatin modification on LINE1 and IAP transposons in PGCs, directly implicating this modification in transposon silencing during DNA hypomethylation. PRMT5 translocates back to the cytoplasm subsequently, to participate in the previously described PIWI-interacting RNA (piRNA) pathway that promotes transposon silencing via de novo DNA remethylation. Thus, PRMT5 is directly involved in genome defense during preimplantation development and in PGCs at the time of global DNA demethylation.','Mol. Cell',2014),(25,'E-GEOD-49555','25515659','Spontaneous development of hepatocellular carcinoma with cancer stem cell properties in PR-SET7-deficient livers.','PR-SET7-mediated histone 4 lysine 20 methylation has been implicated in mitotic condensation, DNA damage response and replication licensing. Here, we show that PR-SET7 function in the liver is pivotal for maintaining genome integrity. Hepatocyte-specific deletion of PR-SET7 in mouse embryos resulted in G2 phase arrest followed by massive cell death and defect in liver organogenesis. Inactivation at postnatal stages caused cell duplication-dependent hepatocyte necrosis, accompanied by inflammation, fibrosis and compensatory growth induction of neighboring hepatocytes and resident ductal progenitor cells. Prolonged necrotic regenerative cycles coupled with oncogenic STAT3 activation led to the spontaneous development of hepatic tumors composed of cells with cancer stem cell characteristics. These include a capacity to self-renew in culture or in xenografts and the ability to differentiate to phenotypically distinct hepatic cells. Hepatocellular carcinoma in PR-SET7-deficient mice displays a cancer stem cell gene signature specified by the co-expression of ductal progenitor markers and oncofetal genes.','EMBO J.',2014);
/*!40000 ALTER TABLE `publication` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `publication_Author`
--

DROP TABLE IF EXISTS `publication_Author`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `publication_Author` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `PubMed` varchar(50) DEFAULT NULL,
  `Author` varchar(50) DEFAULT NULL,
  `AuthorOrder` int(11) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=512 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `publication_Author`
--

LOCK TABLES `publication_Author` WRITE;
/*!40000 ALTER TABLE `publication_Author` DISABLE KEYS */;
INSERT INTO `publication_Author` VALUES (1,'E-GEOD-43631','25017070','Ursula Ehmer',1),(2,'E-GEOD-43631','25017070','Anne-Flore Zmoos',2),(3,'E-GEOD-43631','25017070','Raymond K Auerbach',3),(4,'E-GEOD-43631','25017070','Dedeepya Vaka',4),(5,'E-GEOD-43631','25017070','Atul J Butte',5),(6,'E-GEOD-43631','25017070','Mark A Kay',6),(7,'E-GEOD-43631','25017070','Julien Sage',7),(8,'E-GEOD-46664','24571983','Min Chen',1),(9,'E-GEOD-46664','24571983','Xiaona Wang',2),(10,'E-GEOD-46664','24571983','Yanbo Wang',3),(11,'E-GEOD-46664','24571983','Lianjun Zhang',4),(12,'E-GEOD-46664','24571983','Binyang Xu',5),(13,'E-GEOD-46664','24571983','Limin Lv',6),(14,'E-GEOD-46664','24571983','Xiuhong Cui',7),(15,'E-GEOD-46664','24571983','Wei Li',8),(16,'E-GEOD-46664','24571983','Fei Gao',9),(17,'E-GEOD-47395','25136103','Sanchita Bhatnagar',1),(18,'E-GEOD-47395','25136103','Xiaochun Zhu',2),(19,'E-GEOD-47395','25136103','Jianhong Ou',3),(20,'E-GEOD-47395','25136103','Ling Lin',4),(21,'E-GEOD-47395','25136103','Lynn Chamberlain',5),(22,'E-GEOD-47395','25136103','Lihua J Zhu',6),(23,'E-GEOD-47395','25136103','Narendra Wajapeyee',7),(24,'E-GEOD-47395','25136103','Michael R Green',8),(25,'E-GEOD-49555','25515659','Kostas C Nikolaou',1),(26,'E-GEOD-49555','25515659','Panagiotis Moulos',2),(27,'E-GEOD-49555','25515659','George Chalepakis',3),(28,'E-GEOD-49555','25515659','Pantelis Hatzis',4),(29,'E-GEOD-49555','25515659','Hisanobu Oda',5),(30,'E-GEOD-49555','25515659','Danny Reinberg',6),(31,'E-GEOD-49555','25515659','Iannis Talianidis',7),(32,'E-GEOD-49949','24161570','Abby L Manthey',1),(33,'E-GEOD-49949','24161570','Salil A Lachke',2),(34,'E-GEOD-49949','24161570','Paul G FitzGerald',3),(35,'E-GEOD-49949','24161570','Robert W Mason',4),(36,'E-GEOD-49949','24161570','David A Scheiblin',5),(37,'E-GEOD-49949','24161570','John H McDonald',6),(38,'E-GEOD-49949','24161570','Melinda K Duncan',7),(39,'E-GEOD-51008','25043028','Arianna Sabò',1),(40,'E-GEOD-51008','25043028','Theresia R Kress',2),(41,'E-GEOD-51008','25043028','Mattia Pelizzola',3),(42,'E-GEOD-51008','25043028','Stefano de Pretis',4),(43,'E-GEOD-51008','25043028','Marcin M Gorski',5),(44,'E-GEOD-51008','25043028','Alessandra Tesi',6),(45,'E-GEOD-51008','25043028','Marco J Morelli',7),(46,'E-GEOD-51008','25043028','Pranami Bora',8),(47,'E-GEOD-51008','25043028','Mirko Doni',9),(48,'E-GEOD-51008','25043028','Alessandro Verrecchia',10),(49,'E-GEOD-51008','25043028','Claudia Tonelli',11),(50,'E-GEOD-51008','25043028','Giovanni Fagà',12),(51,'E-GEOD-51008','25043028','Valerio Bianchi',13),(52,'E-GEOD-51008','25043028','Alberto Ronchi',14),(53,'E-GEOD-51008','25043028','Diana Low',15),(54,'E-GEOD-51008','25043028','Heiko Müller',16),(55,'E-GEOD-51008','25043028','Ernesto Guccione',17),(56,'E-GEOD-51008','25043028','Stefano Campaner',18),(57,'E-GEOD-51008','25043028','Bruno Amati',19),(58,'E-GEOD-54154','25008173','Marpadga A Reddy',1),(59,'E-GEOD-54154','25008173','Zhuo Chen',2),(60,'E-GEOD-54154','25008173','Jung Tak Park',3),(61,'E-GEOD-54154','25008173','Mei Wang',4),(62,'E-GEOD-54154','25008173','Linda Lanting',5),(63,'E-GEOD-54154','25008173','Qiang Zhang',6),(64,'E-GEOD-54154','25008173','Kirti Bhatt',7),(65,'E-GEOD-54154','25008173','Amy Leung',8),(66,'E-GEOD-54154','25008173','Xiwei Wu',9),(67,'E-GEOD-54154','25008173','Sumanth Putta',10),(68,'E-GEOD-54154','25008173','Pål Sætrom',11),(69,'E-GEOD-54154','25008173','Sridevi Devaraj',12),(70,'E-GEOD-54154','25008173','Rama Natarajan',13),(71,'E-GEOD-54631','24991935','Yongsheng Bai',1),(72,'E-GEOD-54631','24991935','Justin Hassler',2),(73,'E-GEOD-54631','24991935','Ahdad Ziyar',3),(74,'E-GEOD-54631','24991935','Philip Li',4),(75,'E-GEOD-54631','24991935','Zachary Wright',5),(76,'E-GEOD-54631','24991935','Rajasree Menon',6),(77,'E-GEOD-54631','24991935','Gilbert S Omenn',7),(78,'E-GEOD-54631','24991935','James D Cavalcoli',8),(79,'E-GEOD-54631','24991935','Randal J Kaufman',9),(80,'E-GEOD-54631','24991935','Maureen A Sartor',10),(81,'E-GEOD-55376','24812086','Thomas R Sundermeier',1),(82,'E-GEOD-55376','24812086','Ning Zhang',2),(83,'E-GEOD-55376','24812086','Frans Vinberg',3),(84,'E-GEOD-55376','24812086','Debarshi Mustafi',4),(85,'E-GEOD-55376','24812086','Hideo Kohno',5),(86,'E-GEOD-55376','24812086','Marcin Golczak',6),(87,'E-GEOD-55376','24812086','Xiaodong Bai',7),(88,'E-GEOD-55376','24812086','Akiko Maeda',8),(89,'E-GEOD-55376','24812086','Vladimir J Kefalov',9),(90,'E-GEOD-55376','24812086','Krzysztof Palczewski',10),(91,'E-GEOD-55377','24812086','Thomas R Sundermeier',1),(92,'E-GEOD-55377','24812086','Ning Zhang',2),(93,'E-GEOD-55377','24812086','Frans Vinberg',3),(94,'E-GEOD-55377','24812086','Debarshi Mustafi',4),(95,'E-GEOD-55377','24812086','Hideo Kohno',5),(96,'E-GEOD-55377','24812086','Marcin Golczak',6),(97,'E-GEOD-55377','24812086','Xiaodong Bai',7),(98,'E-GEOD-55377','24812086','Akiko Maeda',8),(99,'E-GEOD-55377','24812086','Vladimir J Kefalov',9),(100,'E-GEOD-55377','24812086','Krzysztof Palczewski',10),(101,'E-GEOD-56018','24745332','Sheng Xiao',1),(102,'E-GEOD-56018','24745332','Nir Yosef',2),(103,'E-GEOD-56018','24745332','Jianfei Yang',3),(104,'E-GEOD-56018','24745332','Yonghui Wang',4),(105,'E-GEOD-56018','24745332','Ling Zhou',5),(106,'E-GEOD-56018','24745332','Chen Zhu',6),(107,'E-GEOD-56018','24745332','Chuan Wu',7),(108,'E-GEOD-56018','24745332','Erkan Baloglu',8),(109,'E-GEOD-56018','24745332','Darby Schmidt',9),(110,'E-GEOD-56018','24745332','Radha Ramesh',10),(111,'E-GEOD-56018','24745332','Mercedes Lobera',11),(112,'E-GEOD-56018','24745332','Mark S Sundrud',12),(113,'E-GEOD-56018','24745332','Pei-Yun Tsai',13),(114,'E-GEOD-56018','24745332','Zhijun Xiang',14),(115,'E-GEOD-56018','24745332','Jinsong Wang',15),(116,'E-GEOD-56018','24745332','Yan Xu',16),(117,'E-GEOD-56018','24745332','Xichen Lin',17),(118,'E-GEOD-56018','24745332','Karsten Kretschmer',18),(119,'E-GEOD-56018','24745332','Peter B Rahl',19),(120,'E-GEOD-56018','24745332','Richard A Young',20),(121,'E-GEOD-56018','24745332','Zhong Zhong',21),(122,'E-GEOD-56018','24745332','David A Hafler',22),(123,'E-GEOD-56018','24745332','Aviv Regev',23),(124,'E-GEOD-56018','24745332','Shomir Ghosh',24),(125,'E-GEOD-56018','24745332','Alexander Marson',25),(126,'E-GEOD-56018','24745332','Vijay K Kuchroo',26),(127,'E-GEOD-56127','25061210','Ryuta Ishimura',1),(128,'E-GEOD-56127','25061210','Gabor Nagy',2),(129,'E-GEOD-56127','25061210','Ivan Dotu',3),(130,'E-GEOD-56127','25061210','Huihao Zhou',4),(131,'E-GEOD-56127','25061210','Xiang-Lei Yang',5),(132,'E-GEOD-56127','25061210','Paul Schimmel',6),(133,'E-GEOD-56127','25061210','Satoru Senju',7),(134,'E-GEOD-56127','25061210','Yasuharu Nishimura',8),(135,'E-GEOD-56127','25061210','Jeffrey H Chuang',9),(136,'E-GEOD-56127','25061210','Susan L Ackerman',10),(137,'E-GEOD-57147','25079327','Margarida A Santos',1),(138,'E-GEOD-57147','25079327','Robert B Faryabi',2),(139,'E-GEOD-57147','25079327','Aysegul V Ergen',3),(140,'E-GEOD-57147','25079327','Amanda M Day',4),(141,'E-GEOD-57147','25079327','Amy Malhowski',5),(142,'E-GEOD-57147','25079327','Andres Canela',6),(143,'E-GEOD-57147','25079327','Masahiro Onozawa',7),(144,'E-GEOD-57147','25079327','Ji-Eun Lee',8),(145,'E-GEOD-57147','25079327','Elsa Callen',9),(146,'E-GEOD-57147','25079327','Paula Gutierrez-Martinez',10),(147,'E-GEOD-57147','25079327','Hua-Tang Chen',11),(148,'E-GEOD-57147','25079327','Nancy Wong',12),(149,'E-GEOD-57147','25079327','Nadia Finkel',13),(150,'E-GEOD-57147','25079327','Aniruddha Deshpande',14),(151,'E-GEOD-57147','25079327','Susan Sharrow',15),(152,'E-GEOD-57147','25079327','Derrick J Rossi',16),(153,'E-GEOD-57147','25079327','Keisuke Ito',17),(154,'E-GEOD-57147','25079327','Kai Ge',18),(155,'E-GEOD-57147','25079327','Peter D Aplan',19),(156,'E-GEOD-57147','25079327','Scott A Armstrong',20),(157,'E-GEOD-57147','25079327','André Nussenzweig',21),(158,'E-GEOD-57396','25002492','Ronan Quéré',1),(159,'E-GEOD-57396','25002492','Laetitia Saint-Paul',2),(160,'E-GEOD-57396','25002492','Virginie Carmignac',3),(161,'E-GEOD-57396','25002492','Romain Z Martin',4),(162,'E-GEOD-57396','25002492','Marie-Lorraine Chrétien',5),(163,'E-GEOD-57396','25002492','Anne Largeot',6),(164,'E-GEOD-57396','25002492','Arlette Hammann',7),(165,'E-GEOD-57396','25002492','Jean-Paul Pais de Barros',8),(166,'E-GEOD-57396','25002492','Jean-Noël Bastie',9),(167,'E-GEOD-57396','25002492','Laurent Delva',10),(168,'E-GEOD-57837','25124037','Fumito Hanihara-Tatsuzawa',1),(169,'E-GEOD-57837','25124037','Hanae Miura',2),(170,'E-GEOD-57837','25124037','Shuhei Kobayashi',3),(171,'E-GEOD-57837','25124037','Takayuki Isagawa',4),(172,'E-GEOD-57837','25124037','Atsushi Okuma',5),(173,'E-GEOD-57837','25124037','Ichiro Manabe',6),(174,'E-GEOD-57837','25124037','Takashi MaruYama',7),(175,'E-GEOD-57959','25146988','Shylaja M Hegde',1),(176,'E-GEOD-57959','25146988','Kiran Srivastava',2),(177,'E-GEOD-57959','25146988','Ekta Tiwary',3),(178,'E-GEOD-57959','25146988','Om P Srivastava',4),(179,'E-GEOD-58498','25002228','Volker Busskamp',1),(180,'E-GEOD-58498','25002228','Jacek Krol',2),(181,'E-GEOD-58498','25002228','Dasha Nelidova',3),(182,'E-GEOD-58498','25002228','Janine Daum',4),(183,'E-GEOD-58498','25002228','Tamas Szikra',5),(184,'E-GEOD-58498','25002228','Ben Tsuda',6),(185,'E-GEOD-58498','25002228','Josephine Jüttner',7),(186,'E-GEOD-58498','25002228','Karl Farrow',8),(187,'E-GEOD-58498','25002228','Brigitte Gross Scherf',9),(188,'E-GEOD-58498','25002228','Claudia Patricia Patino Alvarez',10),(189,'E-GEOD-58498','25002228','Christel Genoud',11),(190,'E-GEOD-58498','25002228','Vithiyanjali Sothilingam',12),(191,'E-GEOD-58498','25002228','Naoyuki Tanimoto',13),(192,'E-GEOD-58498','25002228','Michael Stadler',14),(193,'E-GEOD-58498','25002228','Mathias Seeliger',15),(194,'E-GEOD-58498','25002228','Markus Stoffel',16),(195,'E-GEOD-58498','25002228','Witold Filipowicz',17),(196,'E-GEOD-58498','25002228','Botond Roska',18),(197,'E-GEOD-58905','25126783','Yongqiang Feng',1),(198,'E-GEOD-58905','25126783','Aaron Arvey',2),(199,'E-GEOD-58905','25126783','Takatoshi Chinen',3),(200,'E-GEOD-58905','25126783','Joris van der Veeken',4),(201,'E-GEOD-58905','25126783','Georg Gasteiger',5),(202,'E-GEOD-58905','25126783','Alexander Y Rudensky',6),(203,'E-GEOD-58971','25180063','Nicole Heise',1),(204,'E-GEOD-58971','25180063','Nilushi S De Silva',2),(205,'E-GEOD-58971','25180063','Kathryn Silva',3),(206,'E-GEOD-58971','25180063','Amanda Carette',4),(207,'E-GEOD-58971','25180063','Giorgia Simonetti',5),(208,'E-GEOD-58971','25180063','Manolis Pasparakis',6),(209,'E-GEOD-58971','25180063','Ulf Klein',7),(210,'E-GEOD-58972','25180063','Nicole Heise',1),(211,'E-GEOD-58972','25180063','Nilushi S De Silva',2),(212,'E-GEOD-58972','25180063','Kathryn Silva',3),(213,'E-GEOD-58972','25180063','Amanda Carette',4),(214,'E-GEOD-58972','25180063','Giorgia Simonetti',5),(215,'E-GEOD-58972','25180063','Manolis Pasparakis',6),(216,'E-GEOD-58972','25180063','Ulf Klein',7),(217,'E-GEOD-58973','25180063','Nicole Heise',1),(218,'E-GEOD-58973','25180063','Nilushi S De Silva',2),(219,'E-GEOD-58973','25180063','Kathryn Silva',3),(220,'E-GEOD-58973','25180063','Amanda Carette',4),(221,'E-GEOD-58973','25180063','Giorgia Simonetti',5),(222,'E-GEOD-58973','25180063','Manolis Pasparakis',6),(223,'E-GEOD-58973','25180063','Ulf Klein',7),(224,'E-GEOD-58996','25101683','Michal Mielcarek',1),(225,'E-GEOD-58996','25101683','Linda Inuabasi',2),(226,'E-GEOD-58996','25101683','Marie K Bondulich',3),(227,'E-GEOD-58996','25101683','Thomas Muller',4),(228,'E-GEOD-58996','25101683','Georgina F Osborne',5),(229,'E-GEOD-58996','25101683','Sophie A Franklin',6),(230,'E-GEOD-58996','25101683','Donna L Smith',7),(231,'E-GEOD-58996','25101683','Andreas Neueder',8),(232,'E-GEOD-58996','25101683','Jim Rosinski',9),(233,'E-GEOD-58996','25101683','Ivan Rattray',10),(234,'E-GEOD-58996','25101683','Andrea Protti',11),(235,'E-GEOD-58996','25101683','Gillian P Bates',12),(236,'E-GEOD-59215','25172496','Deirdre D Scripture-Adams',1),(237,'E-GEOD-59215','25172496','Sagar S Damle',2),(238,'E-GEOD-59215','25172496','Long Li',3),(239,'E-GEOD-59215','25172496','Koorosh J Elihu',4),(240,'E-GEOD-59215','25172496','Shuyang Qin',5),(241,'E-GEOD-59215','25172496','Alexandra M Arias',6),(242,'E-GEOD-59215','25172496','Robert R Butler',7),(243,'E-GEOD-59215','25172496','Ameya Champhekar',8),(244,'E-GEOD-59215','25172496','Jingli A Zhang',9),(245,'E-GEOD-59215','25172496','Ellen V Rothenberg',10),(246,'E-GEOD-60968','25269644','Qihuang Jin',1),(247,'E-GEOD-60968','25269644','Lenan Zhuang',2),(248,'E-GEOD-60968','25269644','Binbin Lai',3),(249,'E-GEOD-60968','25269644','Chaochen Wang',4),(250,'E-GEOD-60968','25269644','Wenqian Li',5),(251,'E-GEOD-60968','25269644','Brian Dolan',6),(252,'E-GEOD-60968','25269644','Yue Lu',7),(253,'E-GEOD-60968','25269644','Zhibin Wang',8),(254,'E-GEOD-60968','25269644','Keji Zhao',9),(255,'E-GEOD-60968','25269644','Weiqun Peng',10),(256,'E-GEOD-60968','25269644','Sharon Y R Dent',11),(257,'E-GEOD-60968','25269644','Kai Ge',12),(258,'E-GEOD-61216','25457166','Shinseog Kim',1),(259,'E-GEOD-61216','25457166','Ufuk Günesdogan',2),(260,'E-GEOD-61216','25457166','Jan J Zylicz',3),(261,'E-GEOD-61216','25457166','Jamie A Hackett',4),(262,'E-GEOD-61216','25457166','Delphine Cougot',5),(263,'E-GEOD-61216','25457166','Siqin Bao',6),(264,'E-GEOD-61216','25457166','Caroline Lee',7),(265,'E-GEOD-61216','25457166','Sabine Dietmann',8),(266,'E-GEOD-61216','25457166','George E Allen',9),(267,'E-GEOD-61216','25457166','Roopsha Sengupta',10),(268,'E-GEOD-61216','25457166','M Azim Surani',11);
/*!40000 ALTER TABLE `publication_Author` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `publication_Keyword`
--

DROP TABLE IF EXISTS `publication_Keyword`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `publication_Keyword` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `PubMed` varchar(50) DEFAULT NULL,
  `keyword` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=128 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `publication_Keyword`
--

LOCK TABLES `publication_Keyword` WRITE;
/*!40000 ALTER TABLE `publication_Keyword` DISABLE KEYS */;
INSERT INTO `publication_Keyword` VALUES (1,'E-GEOD-61216','25457166','epigenetic reprogramming'),(2,'E-GEOD-61216','25457166','arginine methyltransferase'),(3,'E-GEOD-61216','25457166','embryotic development'),(4,'E-GEOD-61216','25457166','sterility'),(5,'E-GEOD-61216','25457166','infertility'),(6,'E-GEOD-61216','25457166','LINE1 and IAP transposons'),(7,'E-GEOD-60968','25269644','Gcn5/PCAF'),(8,'E-GEOD-60968','25269644','H3K9ac'),(9,'E-GEOD-60968','25269644','TBK1'),(10,'E-GEOD-60968','25269644','innate immune signaling'),(11,'E-GEOD-60968','25269644','interferon‐β'),(12,'E-GEOD-59215','25172496','GATA-3 deletion'),(13,'E-GEOD-59215','25172496','T-cell development'),(14,'E-GEOD-58996','25101683','cardiac remodelling'),(15,'E-GEOD-58996','25101683','Huntington\'s disease'),(16,'E-GEOD-58996','25101683','R6/2 transgenic HdhQ150 KI'),(17,'E-GEOD-58973','25180063','lymphomagenesis'),(18,'E-GEOD-58973','25180063','germinal centers'),(19,'E-GEOD-58973','25180063','anti-CD40 and anti-IgM stimulus'),(20,'E-GEOD-58972','25180063','lymphomagenesis'),(21,'E-GEOD-58972','25180063','germinal centers'),(22,'E-GEOD-58972','25180063','anti-CD40 and anti-IgM stimulus'),(23,'E-GEOD-58971','25180063','lymphomagenesis'),(24,'E-GEOD-58971','25180063','germinal centers'),(25,'E-GEOD-58971','25180063','anti-CD40 and anti-IgM stimulus'),(26,'E-GEOD-58905','25126783','reulgatory T cells'),(27,'E-GEOD-58905','25126783','inheritance'),(28,'E-GEOD-58498','25002228','blindness'),(29,'E-GEOD-58498','25002228','DiGeorge Syndrome'),(30,'E-GEOD-58498','25002228','DGCR8'),(31,'E-GEOD-58498','25002228','MicroRNA maintenance of cone outer segments'),(32,'E-GEOD-57959','25146988','alpha A crystallin'),(33,'E-GEOD-57959','25146988','cortical opacity'),(34,'E-GEOD-57959','25146988','deamidation'),(35,'E-GEOD-57959','25146988','denucleation'),(36,'E-GEOD-57837','25124037','Activation-induced Cytidine Deaminase (AID)'),(37,'E-GEOD-57837','25124037','B Cell'),(38,'E-GEOD-57837','25124037','Cell Differentiation'),(39,'E-GEOD-57837','25124037','Cellular Immune Response'),(40,'E-GEOD-57837','25124037','Immunology'),(41,'E-GEOD-57837','25124037','IκB-z'),(42,'E-GEOD-57837','25124037','Toll-like Receptor (TLR)'),(43,'E-GEOD-57837','25124037','Transcription Factor'),(44,'E-GEOD-57396','25002492','Tif1γ regulates the TGF-β1 receptor and promotes p'),(45,'E-GEOD-57147','25079327','myeloid differentiation'),(46,'E-GEOD-57147','25079327','histone methyl-transferase MLL4'),(47,'E-GEOD-57147','25079327','acute myeloid leukemia'),(48,'E-GEOD-56127','25061210','Molecular biology'),(49,'E-GEOD-56127','25061210','Ribosome rescue'),(50,'E-GEOD-56127','25061210','neurodegeneration'),(51,'E-GEOD-56127','25061210','loss of GTPBP2'),(52,'E-GEOD-56127','25061210','Pelota'),(53,'E-GEOD-56018','24745332','RORγt'),(54,'E-GEOD-56018','24745332','T helper 17'),(55,'E-GEOD-56018','24745332','T cell differentiation'),(56,'E-GEOD-55377','24812086','age-related macular degeneration.'),(57,'E-GEOD-55377','24812086','cell survival'),(58,'E-GEOD-55377','24812086','conditional knockout'),(59,'E-GEOD-55377','24812086','microRNA'),(60,'E-GEOD-55377','24812086','retina'),(61,'E-GEOD-55376','24812086','age-related macular degeneration.'),(62,'E-GEOD-55376','24812086','cell survival'),(63,'E-GEOD-55376','24812086','conditional knockout'),(64,'E-GEOD-55376','24812086','microRNA'),(65,'E-GEOD-55376','24812086','retina'),(66,'E-GEOD-54631','24991935','endoribonuclease'),(67,'E-GEOD-54631','24991935','ER stress'),(68,'E-GEOD-54631','24991935','Ire1α'),(69,'E-GEOD-54631','24991935','non-canonical splice junction site discovery'),(70,'E-GEOD-54154','25008173','microphage transcriptome sequencing'),(71,'E-GEOD-54154','25008173','diabetes and complications'),(72,'E-GEOD-54154','25008173','inflammation'),(73,'E-GEOD-54154','25008173','GM-CSF (BMGM)'),(74,'E-GEOD-54154','25008173','MCSF-1 (BMMC) '),(75,'E-GEOD-51008','25043028','amplification in cells'),(76,'E-GEOD-51008','25043028','transcriptional amplifiers'),(77,'E-GEOD-51008','25043028','Myc activity'),(78,'E-GEOD-51008','25043028','proto-ongogene'),(79,'E-GEOD-51008','25043028','lymphomagenesis'),(80,'E-GEOD-49949','24161570','Ectodermal cell fate'),(81,'E-GEOD-49949','24161570','Lens development'),(82,'E-GEOD-49949','24161570','Sip1'),(83,'E-GEOD-49949','24161570','Zeb2'),(84,'E-GEOD-49555','25515659','ductal progenitors'),(85,'E-GEOD-49555','25515659','hepatocellular carcinoma'),(86,'E-GEOD-49555','25515659','histone methylase'),(87,'E-GEOD-47395','25136103','MECP2'),(88,'E-GEOD-47395','25136103','RNA FISH'),(89,'E-GEOD-47395','25136103','RNA-seq'),(90,'E-GEOD-46664','24571983','Leydig cells'),(91,'E-GEOD-46664','24571983','gene regulation'),(92,'E-GEOD-46664','24571983','spermatogenesis'),(93,'E-GEOD-46664','24571983','testis'),(94,'E-GEOD-46664','24571983','testosterone'),(95,'E-GEOD-43631','25017070','inactivation of Rb pathway');
/*!40000 ALTER TABLE `publication_Keyword` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `tissue`
--

DROP TABLE IF EXISTS `tissue`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `tissue` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `ArrayExpress` varchar(20) DEFAULT NULL,
  `Tissue` varchar(50) DEFAULT NULL,
  `TissueID` varchar(50) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=64 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `tissue`
--

LOCK TABLES `tissue` WRITE;
/*!40000 ALTER TABLE `tissue` DISABLE KEYS */;
INSERT INTO `tissue` VALUES (1,'E-GEOD-61216','embryonic germ cell','BTO:0005319'),(2,'E-GEOD-61216','preimplantation embryos',''),(3,'E-GEOD-60968','embryonic fibroblast','BTO:0004725'),(4,'E-GEOD-60968','SKBR3 cell','BTO:0002419'),(5,'E-GEOD-59215','thymocyte','BTO:0001372'),(6,'E-GEOD-58996','heart','BTO:0000562'),(7,'E-GEOD-58973','pro-B-lymphocyte','BTO:0003104'),(8,'E-GEOD-58972','pro-B-lymphocyte','BTO:0003104'),(9,'E-GEOD-58971','pro-B-lymphocyte','BTO:0003104'),(10,'E-GEOD-58905','regulatory T-lymphocyte','BTO:0004520'),(11,'E-GEOD-58498','eye','BTO:0000439'),(12,'E-GEOD-57959','eye','BTO:0000439'),(13,'E-GEOD-57837','T-lymphocyte','BTO:0000782'),(14,'E-GEOD-57837','B-lymphocyte','BTO:0000776'),(15,'E-GEOD-57396','hematopoietic stem cell','BTO:0000725'),(16,'E-GEOD-57147','hematopoietic stem cell','BTO:0000725'),(17,'E-GEOD-57147','hematopoietic stem cell, blast cell','BTO:0000125'),(18,'E-GEOD-57147','hematopoietic stem cell, blast cell, myeloid','BTO:0001441'),(19,'E-GEOD-57147','LSK bone marrow cells',''),(20,'E-GEOD-56127','cerebellum','BTO:0000232'),(21,'E-GEOD-56018','Th17 cell','BTO:0005655'),(22,'E-GEOD-55377','retina','BTO:0001175'),(23,'E-GEOD-55376','retina','BTO:0001175'),(24,'E-GEOD-54631','embryonic fibroblast','BTO:0004725'),(25,'E-GEOD-54631','SK-BR-3 cell','BTO:0002419'),(26,'E-GEOD-54154','bone marrow-derived macrophage','BTO:0004732'),(27,'E-GEOD-51008','embryonic fibroblast','BTO:0004725'),(28,'E-GEOD-49949','lens','BTO:0000723'),(29,'E-GEOD-49555','Hepatocyte','BTO:0000575'),(30,'E-GEOD-49555','hematopoietic stem cell','BTO:0000725'),(31,'E-GEOD-47395','embryonic fibroblast','BTO:0004725'),(32,'E-GEOD-46664','Sertoli cell','BTO:0001238'),(33,'E-GEOD-46664','Leydig cells','BTO:0000755'),(34,'E-GEOD-43631','hepatocyte','BTO:0000575');
/*!40000 ALTER TABLE `tissue` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `user`
--

DROP TABLE IF EXISTS `user`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `user` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(50) NOT NULL,
  `email` varchar(50) NOT NULL,
  `institution` varchar(50) NOT NULL,
  `pwhash` varchar(200) NOT NULL,
  `downloadedtimes` int(11) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `user`
--

LOCK TABLES `user` WRITE;
/*!40000 ALTER TABLE `user` DISABLE KEYS */;
/*!40000 ALTER TABLE `user` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2015-06-15 22:18:25
